1566306730886	Sync.Service	INFO	Loading Weave 1.47.0
1566306730892	Sync.Engine.Clients	DEBUG	Engine initialized
1566306730896	Sync.Engine.Clients	DEBUG	Resetting clients last sync time
1566306730907	Sync.Engine.Bookmarks	DEBUG	Engine initialized
1566306730916	Sync.Engine.Forms	DEBUG	Engine initialized
1566306730929	Sync.Engine.History	DEBUG	Engine initialized
1566306730938	Sync.Engine.Passwords	DEBUG	Engine initialized
1566306730947	Sync.Engine.Prefs	DEBUG	Engine initialized
1566306730953	Sync.Engine.Tabs	DEBUG	Engine initialized
1566306730954	Sync.Engine.Tabs	DEBUG	Resetting tabs last sync time
1566306730967	Sync.Engine.Addons	DEBUG	Engine initialized
1566306730970	Sync.Service	INFO	Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:45.0) Gecko/20100101 Firefox/45.0
1566306730977	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1566306730983	Sync.Status	DEBUG	Status.service: success.status_ok => success.status_ok
1566306730984	Sync.Status	DEBUG	Status.service: success.status_ok => success.status_ok
1566306730984	Sync.AddonsReconciler	INFO	Registering as Add-on Manager listener.
1566306730985	Sync.AddonsReconciler	DEBUG	Adding change listener.
1566306730986	Sync.Tracker.History	INFO	Adding Places observer.
1566306730994	FirefoxAccounts	DEBUG	startVerifiedCheck: false
1566306730999	FirefoxAccounts	DEBUG	entering pollEmailStatus: timer
1566306731021	browserwindow.syncui	DEBUG	observed: weave:service:ready
1566306731027	Sync.Status	DEBUG	Status.service: success.status_ok => success.status_ok
1566306731031	FirefoxAccounts	DEBUG	startVerifiedCheck: false
1566306731032	FirefoxAccounts	DEBUG	startVerifiedCheck: false
1566306731033	browserwindow.syncui	DEBUG	_loginFailed has sync state=success.login
1566306731035	FirefoxAccounts	DEBUG	startVerifiedCheck: false
1566306731036	FirefoxAccounts	DEBUG	startVerifiedCheck: false
1566306731037	browserwindow.syncui	DEBUG	_loginFailed has sync state=success.login
1566306731041	Sync.Tracker.Clients	WARN	Changed IDs file clients contains non-object value.
1566306731046	Hawk	WARN	hawk request error: [Exception... "NS_ERROR_UNKNOWN_HOST"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "JS frame :: resource://services-common/rest.js :: onStopRequest :: line 465"  data: no] Stack trace: onStopRequest()@resource://services-common/rest.js:465
1566306731047	Sync.RESTResponse	DEBUG	Caught exception fetching HTTP status code:[Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatus]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: RESTResponse.prototype.status :: line 643"  data: no] Stack trace: RESTResponse.prototype.status()@resource://services-common/rest.js:643 < _onComplete()@resource://services-common/hawkclient.js:218 < onComplete()@resource://services-common/hawkclient.js:275 < onStopRequest()@resource://services-common/rest.js:467
1566306731048	Sync.RESTResponse	DEBUG	Caught exception fetching HTTP status text:[Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatusText]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: RESTResponse.prototype.statusText :: line 659"  data: no] Stack trace: RESTResponse.prototype.statusText()@resource://services-common/rest.js:659 < _onComplete()@resource://services-common/hawkclient.js:221 < onComplete()@resource://services-common/hawkclient.js:275 < onStopRequest()@resource://services-common/rest.js:467
1566306731048	Hawk	DEBUG	(Response) /recovery_email/status: code: null - Status text: null
1566306731049	Sync.RESTResponse	TRACE	Processing response headers.
1566306731049	Sync.RESTResponse	DEBUG	Caught exception processing response headers:[Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.visitResponseHeaders]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: RESTResponse.prototype.headers :: line 693"  data: no] Stack trace: RESTResponse.prototype.headers()@resource://services-common/rest.js:693 < this.HawkClient.prototype._maybeNotifyBackoff()@resource://services-common/hawkclient.js:315 < _onComplete()@resource://services-common/hawkclient.js:228 < onComplete()@resource://services-common/hawkclient.js:275 < onStopRequest()@resource://services-common/rest.js:467
1566306731049	Sync.RESTResponse	TRACE	Processing response headers.
1566306731050	Sync.RESTResponse	DEBUG	Caught exception processing response headers:[Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.visitResponseHeaders]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: RESTResponse.prototype.headers :: line 693"  data: no] Stack trace: RESTResponse.prototype.headers()@resource://services-common/rest.js:693 < this.HawkClient.prototype._maybeNotifyBackoff()@resource://services-common/hawkclient.js:315 < _onComplete()@resource://services-common/hawkclient.js:229 < onComplete()@resource://services-common/hawkclient.js:275 < onStopRequest()@resource://services-common/rest.js:467
1566306731050	Sync.RESTResponse	DEBUG	Caught exception fetching HTTP status text:[Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatusText]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: RESTResponse.prototype.statusText :: line 659"  data: no] Stack trace: RESTResponse.prototype.statusText()@resource://services-common/rest.js:659 < this.HawkClient.prototype._constructError()@resource://services-common/hawkclient.js:121 < _onComplete()@resource://services-common/hawkclient.js:234 < onComplete()@resource://services-common/hawkclient.js:275 < onStopRequest()@resource://services-common/rest.js:467
1566306731050	Sync.RESTResponse	DEBUG	Caught exception fetching HTTP status code:[Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatus]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: RESTResponse.prototype.status :: line 643"  data: no] Stack trace: RESTResponse.prototype.status()@resource://services-common/rest.js:643 < this.HawkClient.prototype._constructError()@resource://services-common/hawkclient.js:122 < _onComplete()@resource://services-common/hawkclient.js:234 < onComplete()@resource://services-common/hawkclient.js:275 < onStopRequest()@resource://services-common/rest.js:467
1566306731051	Sync.RESTResponse	DEBUG	Caught exception fetching HTTP status code:[Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatus]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: RESTResponse.prototype.status :: line 643"  data: no] Stack trace: RESTResponse.prototype.status()@resource://services-common/rest.js:643 < this.HawkClient.prototype._constructError()@resource://services-common/hawkclient.js:123 < _onComplete()@resource://services-common/hawkclient.js:234 < onComplete()@resource://services-common/hawkclient.js:275 < onStopRequest()@resource://services-common/rest.js:467
1566306731051	Sync.RESTResponse	TRACE	Processing response headers.
1566306731051	Sync.RESTResponse	DEBUG	Caught exception processing response headers:[Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.visitResponseHeaders]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: RESTResponse.prototype.headers :: line 693"  data: no] Stack trace: RESTResponse.prototype.headers()@resource://services-common/rest.js:693 < this.HawkClient.prototype._constructError()@resource://services-common/hawkclient.js:128 < _onComplete()@resource://services-common/hawkclient.js:234 < onComplete()@resource://services-common/hawkclient.js:275 < onStopRequest()@resource://services-common/rest.js:467
1566306731052	FirefoxAccounts	ERROR	error GETing /recovery_email/status: {"error":{},"errorString":"[Exception... \"NS_ERROR_UNKNOWN_HOST\"  nsresult: \"0x804b001e (NS_ERROR_UNKNOWN_HOST)\"  location: \"JS frame :: resource://services-common/rest.js :: onStopRequest :: line 465\"  data: no]","message":null,"code":null,"errno":null}
1566306731053	FirefoxAccounts	DEBUG	polling with timeout = 5000
1566306731061	Sync.Tracker.Bookmarks	WARN	Changed IDs file bookmarks contains non-object value.
1566306731066	Sync.Tracker.Forms	WARN	Changed IDs file forms contains non-object value.
1566306731069	Sync.Tracker.History	WARN	Changed IDs file history contains non-object value.
1566306731075	Sync.Tracker.Passwords	WARN	Changed IDs file passwords contains non-object value.
1566306731083	Sync.Tracker.Addons	WARN	Changed IDs file addons contains non-object value.
1566306736054	FirefoxAccounts	DEBUG	entering pollEmailStatus: timer
1566306736066	Hawk	WARN	hawk request error: [Exception... "NS_ERROR_UNKNOWN_HOST"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "JS frame :: resource://services-common/rest.js :: onStopRequest :: line 465"  data: no] Stack trace: onStopRequest()@resource://services-common/rest.js:465
1566306736067	Sync.RESTResponse	DEBUG	Caught exception fetching HTTP status code:[Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatus]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: RESTResponse.prototype.status :: line 643"  data: no] Stack trace: RESTResponse.prototype.status()@resource://services-common/rest.js:643 < _onComplete()@resource://services-common/hawkclient.js:218 < onComplete()@resource://services-common/hawkclient.js:275 < onStopRequest()@resource://services-common/rest.js:467
1566306736068	Sync.RESTResponse	DEBUG	Caught exception fetching HTTP status text:[Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatusText]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: RESTResponse.prototype.statusText :: line 659"  data: no] Stack trace: RESTResponse.prototype.statusText()@resource://services-common/rest.js:659 < _onComplete()@resource://services-common/hawkclient.js:221 < onComplete()@resource://services-common/hawkclient.js:275 < onStopRequest()@resource://services-common/rest.js:467
1566306736068	Hawk	DEBUG	(Response) /recovery_email/status: code: null - Status text: null
1566306736068	Sync.RESTResponse	TRACE	Processing response headers.
1566306736069	Sync.RESTResponse	DEBUG	Caught exception processing response headers:[Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.visitResponseHeaders]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: RESTResponse.prototype.headers :: line 693"  data: no] Stack trace: RESTResponse.prototype.headers()@resource://services-common/rest.js:693 < this.HawkClient.prototype._maybeNotifyBackoff()@resource://services-common/hawkclient.js:315 < _onComplete()@resource://services-common/hawkclient.js:228 < onComplete()@resource://services-common/hawkclient.js:275 < onStopRequest()@resource://services-common/rest.js:467
1566306736069	Sync.RESTResponse	TRACE	Processing response headers.
1566306736070	Sync.RESTResponse	DEBUG	Caught exception processing response headers:[Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.visitResponseHeaders]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: RESTResponse.prototype.headers :: line 693"  data: no] Stack trace: RESTResponse.prototype.headers()@resource://services-common/rest.js:693 < this.HawkClient.prototype._maybeNotifyBackoff()@resource://services-common/hawkclient.js:315 < _onComplete()@resource://services-common/hawkclient.js:229 < onComplete()@resource://services-common/hawkclient.js:275 < onStopRequest()@resource://services-common/rest.js:467
1566306736070	Sync.RESTResponse	DEBUG	Caught exception fetching HTTP status text:[Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatusText]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: RESTResponse.prototype.statusText :: line 659"  data: no] Stack trace: RESTResponse.prototype.statusText()@resource://services-common/rest.js:659 < this.HawkClient.prototype._constructError()@resource://services-common/hawkclient.js:121 < _onComplete()@resource://services-common/hawkclient.js:234 < onComplete()@resource://services-common/hawkclient.js:275 < onStopRequest()@resource://services-common/rest.js:467
1566306736070	Sync.RESTResponse	DEBUG	Caught exception fetching HTTP status code:[Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatus]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: RESTResponse.prototype.status :: line 643"  data: no] Stack trace: RESTResponse.prototype.status()@resource://services-common/rest.js:643 < this.HawkClient.prototype._constructError()@resource://services-common/hawkclient.js:122 < _onComplete()@resource://services-common/hawkclient.js:234 < onComplete()@resource://services-common/hawkclient.js:275 < onStopRequest()@resource://services-common/rest.js:467
1566306736071	Sync.RESTResponse	DEBUG	Caught exception fetching HTTP status code:[Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatus]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: RESTResponse.prototype.status :: line 643"  data: no] Stack trace: RESTResponse.prototype.status()@resource://services-common/rest.js:643 < this.HawkClient.prototype._constructError()@resource://services-common/hawkclient.js:123 < _onComplete()@resource://services-common/hawkclient.js:234 < onComplete()@resource://services-common/hawkclient.js:275 < onStopRequest()@resource://services-common/rest.js:467
1566306736071	Sync.RESTResponse	TRACE	Processing response headers.
1566306736071	Sync.RESTResponse	DEBUG	Caught exception processing response headers:[Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.visitResponseHeaders]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: RESTResponse.prototype.headers :: line 693"  data: no] Stack trace: RESTResponse.prototype.headers()@resource://services-common/rest.js:693 < this.HawkClient.prototype._constructError()@resource://services-common/hawkclient.js:128 < _onComplete()@resource://services-common/hawkclient.js:234 < onComplete()@resource://services-common/hawkclient.js:275 < onStopRequest()@resource://services-common/rest.js:467
1566306736072	FirefoxAccounts	ERROR	error GETing /recovery_email/status: {"error":{},"errorString":"[Exception... \"NS_ERROR_UNKNOWN_HOST\"  nsresult: \"0x804b001e (NS_ERROR_UNKNOWN_HOST)\"  location: \"JS frame :: resource://services-common/rest.js :: onStopRequest :: line 465\"  data: no]","message":null,"code":null,"errno":null}
1566306736073	FirefoxAccounts	DEBUG	polling with timeout = 5000
1566306738028	Sync.Status	DEBUG	Status.service: success.status_ok => success.status_ok
1566306738030	Sync.Service	DEBUG	User-Agent: Firefox/45.0.2 FxSync/1.47.0.20160414065514.
1566306738031	Sync.Service	INFO	Starting sync at 2019-08-20 14:12:18
1566306738031	Sync.Service	DEBUG	In sync: should login.
1566306738032	browserwindow.syncui	DEBUG	observed: weave:service:login:start
1566306738035	Sync.Status	DEBUG	Status.service: success.status_ok => success.status_ok
1566306738035	Sync.Status	DEBUG	Status.service: success.status_ok => success.status_ok
1566306738035	Sync.Service	INFO	Logging in the user.
1566306738040	FirefoxAccounts	DEBUG	startVerifiedCheck: false
1566306738041	FirefoxAccounts	DEBUG	startVerifiedCheck: false
1566306738042	Sync.BrowserIDManager	INFO	Waiting for user to be verified.
1566306738042	browserwindow.syncui	DEBUG	_loginFailed has sync state=success.login
1566306738044	FirefoxAccounts	DEBUG	startVerifiedCheck: false
1566306738045	FirefoxAccounts	DEBUG	startVerifiedCheck: false
1566306738046	browserwindow.syncui	DEBUG	_loginFailed has sync state=success.login
1566306741073	FirefoxAccounts	DEBUG	entering pollEmailStatus: timer
1566306741082	Hawk	WARN	hawk request error: [Exception... "NS_ERROR_UNKNOWN_HOST"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "JS frame :: resource://services-common/rest.js :: onStopRequest :: line 465"  data: no] Stack trace: onStopRequest()@resource://services-common/rest.js:465 < waitForSyncCallback()@resource://services-common/async.js:102 < makeSpinningCallback/callback.wait()@resource://services-common/async.js:156 < onNotify()@resource://gre/modules/services-sync/service.js:1009 < WrappedNotify()@resource://services-sync/util.js:148 < WrappedLock()@resource://services-sync/util.js:103 < WrappedCatch()@resource://services-sync/util.js:77 < login()@resource://gre/modules/services-sync/service.js:1031 < sync/<()@resource://gre/modules/services-sync/service.js:1283 < WrappedCatch()@resource://services-sync/util.js:77 < sync()@resource://gre/modules/services-sync/service.js:1279
1566306741084	Sync.RESTResponse	DEBUG	Caught exception fetching HTTP status code:[Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatus]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: RESTResponse.prototype.status :: line 643"  data: no] Stack trace: RESTResponse.prototype.status()@resource://services-common/rest.js:643 < _onComplete()@resource://services-common/hawkclient.js:218 < onComplete()@resource://services-common/hawkclient.js:275 < onStopRequest()@resource://services-common/rest.js:467 < waitForSyncCallback()@resource://services-common/async.js:102 < makeSpinningCallback/callback.wait()@resource://services-common/async.js:156 < onNotify()@resource://gre/modules/services-sync/service.js:1009 < WrappedNotify()@resource://services-sync/util.js:148 < WrappedLock()@resource://services-sync/util.js:103 < WrappedCatch()@resource://services-sync/util.js:77 < login()@resource://gre/modules/services-sync/service.js:1031 < sync/<()@resource://gre/modules/services-sync/service.js:1283 < WrappedCatch()@resource://services-sync/util.js:77 < sync()@resource://gre/modules/services-sync/service.js:1279
1566306741085	Sync.RESTResponse	DEBUG	Caught exception fetching HTTP status text:[Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatusText]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: RESTResponse.prototype.statusText :: line 659"  data: no] Stack trace: RESTResponse.prototype.statusText()@resource://services-common/rest.js:659 < _onComplete()@resource://services-common/hawkclient.js:221 < onComplete()@resource://services-common/hawkclient.js:275 < onStopRequest()@resource://services-common/rest.js:467 < waitForSyncCallback()@resource://services-common/async.js:102 < makeSpinningCallback/callback.wait()@resource://services-common/async.js:156 < onNotify()@resource://gre/modules/services-sync/service.js:1009 < WrappedNotify()@resource://services-sync/util.js:148 < WrappedLock()@resource://services-sync/util.js:103 < WrappedCatch()@resource://services-sync/util.js:77 < login()@resource://gre/modules/services-sync/service.js:1031 < sync/<()@resource://gre/modules/services-sync/service.js:1283 < WrappedCatch()@resource://services-sync/util.js:77 < sync()@resource://gre/modules/services-sync/service.js:1279
1566306741085	Hawk	DEBUG	(Response) /recovery_email/status: code: null - Status text: null
1566306741085	Sync.RESTResponse	TRACE	Processing response headers.
1566306741086	Sync.RESTResponse	DEBUG	Caught exception processing response headers:[Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.visitResponseHeaders]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: RESTResponse.prototype.headers :: line 693"  data: no] Stack trace: RESTResponse.prototype.headers()@resource://services-common/rest.js:693 < this.HawkClient.prototype._maybeNotifyBackoff()@resource://services-common/hawkclient.js:315 < _onComplete()@resource://services-common/hawkclient.js:228 < onComplete()@resource://services-common/hawkclient.js:275 < onStopRequest()@resource://services-common/rest.js:467 < waitForSyncCallback()@resource://services-common/async.js:102 < makeSpinningCallback/callback.wait()@resource://services-common/async.js:156 < onNotify()@resource://gre/modules/services-sync/service.js:1009 < WrappedNotify()@resource://services-sync/util.js:148 < WrappedLock()@resource://services-sync/util.js:103 < WrappedCatch()@resource://services-sync/util.js:77 < login()@resource://gre/modules/services-sync/service.js:1031 < sync/<()@resource://gre/modules/services-sync/service.js:1283 < WrappedCatch()@resource://services-sync/util.js:77 < sync()@resource://gre/modules/services-sync/service.js:1279
1566306741086	Sync.RESTResponse	TRACE	Processing response headers.
1566306741087	Sync.RESTResponse	DEBUG	Caught exception processing response headers:[Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.visitResponseHeaders]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: RESTResponse.prototype.headers :: line 693"  data: no] Stack trace: RESTResponse.prototype.headers()@resource://services-common/rest.js:693 < this.HawkClient.prototype._maybeNotifyBackoff()@resource://services-common/hawkclient.js:315 < _onComplete()@resource://services-common/hawkclient.js:229 < onComplete()@resource://services-common/hawkclient.js:275 < onStopRequest()@resource://services-common/rest.js:467 < waitForSyncCallback()@resource://services-common/async.js:102 < makeSpinningCallback/callback.wait()@resource://services-common/async.js:156 < onNotify()@resource://gre/modules/services-sync/service.js:1009 < WrappedNotify()@resource://services-sync/util.js:148 < WrappedLock()@resource://services-sync/util.js:103 < WrappedCatch()@resource://services-sync/util.js:77 < login()@resource://gre/modules/services-sync/service.js:1031 < sync/<()@resource://gre/modules/services-sync/service.js:1283 < WrappedCatch()@resource://services-sync/util.js:77 < sync()@resource://gre/modules/services-sync/service.js:1279
1566306741088	Sync.RESTResponse	DEBUG	Caught exception fetching HTTP status text:[Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatusText]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: RESTResponse.prototype.statusText :: line 659"  data: no] Stack trace: RESTResponse.prototype.statusText()@resource://services-common/rest.js:659 < this.HawkClient.prototype._constructError()@resource://services-common/hawkclient.js:121 < _onComplete()@resource://services-common/hawkclient.js:234 < onComplete()@resource://services-common/hawkclient.js:275 < onStopRequest()@resource://services-common/rest.js:467 < waitForSyncCallback()@resource://services-common/async.js:102 < makeSpinningCallback/callback.wait()@resource://services-common/async.js:156 < onNotify()@resource://gre/modules/services-sync/service.js:1009 < WrappedNotify()@resource://services-sync/util.js:148 < WrappedLock()@resource://services-sync/util.js:103 < WrappedCatch()@resource://services-sync/util.js:77 < login()@resource://gre/modules/services-sync/service.js:1031 < sync/<()@resource://gre/modules/services-sync/service.js:1283 < WrappedCatch()@resource://services-sync/util.js:77 < sync()@resource://gre/modules/services-sync/service.js:1279
1566306741089	Sync.RESTResponse	DEBUG	Caught exception fetching HTTP status code:[Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatus]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: RESTResponse.prototype.status :: line 643"  data: no] Stack trace: RESTResponse.prototype.status()@resource://services-common/rest.js:643 < this.HawkClient.prototype._constructError()@resource://services-common/hawkclient.js:122 < _onComplete()@resource://services-common/hawkclient.js:234 < onComplete()@resource://services-common/hawkclient.js:275 < onStopRequest()@resource://services-common/rest.js:467 < waitForSyncCallback()@resource://services-common/async.js:102 < makeSpinningCallback/callback.wait()@resource://services-common/async.js:156 < onNotify()@resource://gre/modules/services-sync/service.js:1009 < WrappedNotify()@resource://services-sync/util.js:148 < WrappedLock()@resource://services-sync/util.js:103 < WrappedCatch()@resource://services-sync/util.js:77 < login()@resource://gre/modules/services-sync/service.js:1031 < sync/<()@resource://gre/modules/services-sync/service.js:1283 < WrappedCatch()@resource://services-sync/util.js:77 < sync()@resource://gre/modules/services-sync/service.js:1279
1566306741090	Sync.RESTResponse	DEBUG	Caught exception fetching HTTP status code:[Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatus]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: RESTResponse.prototype.status :: line 643"  data: no] Stack trace: RESTResponse.prototype.status()@resource://services-common/rest.js:643 < this.HawkClient.prototype._constructError()@resource://services-common/hawkclient.js:123 < _onComplete()@resource://services-common/hawkclient.js:234 < onComplete()@resource://services-common/hawkclient.js:275 < onStopRequest()@resource://services-common/rest.js:467 < waitForSyncCallback()@resource://services-common/async.js:102 < makeSpinningCallback/callback.wait()@resource://services-common/async.js:156 < onNotify()@resource://gre/modules/services-sync/service.js:1009 < WrappedNotify()@resource://services-sync/util.js:148 < WrappedLock()@resource://services-sync/util.js:103 < WrappedCatch()@resource://services-sync/util.js:77 < login()@resource://gre/modules/services-sync/service.js:1031 < sync/<()@resource://gre/modules/services-sync/service.js:1283 < WrappedCatch()@resource://services-sync/util.js:77 < sync()@resource://gre/modules/services-sync/service.js:1279
1566306741091	Sync.RESTResponse	TRACE	Processing response headers.
1566306741092	Sync.RESTResponse	DEBUG	Caught exception processing response headers:[Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.visitResponseHeaders]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: RESTResponse.prototype.headers :: line 693"  data: no] Stack trace: RESTResponse.prototype.headers()@resource://services-common/rest.js:693 < this.HawkClient.prototype._constructError()@resource://services-common/hawkclient.js:128 < _onComplete()@resource://services-common/hawkclient.js:234 < onComplete()@resource://services-common/hawkclient.js:275 < onStopRequest()@resource://services-common/rest.js:467 < waitForSyncCallback()@resource://services-common/async.js:102 < makeSpinningCallback/callback.wait()@resource://services-common/async.js:156 < onNotify()@resource://gre/modules/services-sync/service.js:1009 < WrappedNotify()@resource://services-sync/util.js:148 < WrappedLock()@resource://services-sync/util.js:103 < WrappedCatch()@resource://services-sync/util.js:77 < login()@resource://gre/modules/services-sync/service.js:1031 < sync/<()@resource://gre/modules/services-sync/service.js:1283 < WrappedCatch()@resource://services-sync/util.js:77 < sync()@resource://gre/modules/services-sync/service.js:1279
1566306741092	FirefoxAccounts	ERROR	error GETing /recovery_email/status: {"error":{},"errorString":"[Exception... \"NS_ERROR_UNKNOWN_HOST\"  nsresult: \"0x804b001e (NS_ERROR_UNKNOWN_HOST)\"  location: \"JS frame :: resource://services-common/rest.js :: onStopRequest :: line 465\"  data: no]","message":null,"code":null,"errno":null}
1566306741093	FirefoxAccounts	DEBUG	polling with timeout = 5000
1566306746094	FirefoxAccounts	DEBUG	entering pollEmailStatus: timer
1566306746111	Hawk	WARN	hawk request error: [Exception... "NS_ERROR_UNKNOWN_HOST"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "JS frame :: resource://services-common/rest.js :: onStopRequest :: line 465"  data: no] Stack trace: onStopRequest()@resource://services-common/rest.js:465 < waitForSyncCallback()@resource://services-common/async.js:102 < makeSpinningCallback/callback.wait()@resource://services-common/async.js:156 < onNotify()@resource://gre/modules/services-sync/service.js:1009 < WrappedNotify()@resource://services-sync/util.js:148 < WrappedLock()@resource://services-sync/util.js:103 < WrappedCatch()@resource://services-sync/util.js:77 < login()@resource://gre/modules/services-sync/service.js:1031 < sync/<()@resource://gre/modules/services-sync/service.js:1283 < WrappedCatch()@resource://services-sync/util.js:77 < sync()@resource://gre/modules/services-sync/service.js:1279
1566306746113	Sync.RESTResponse	DEBUG	Caught exception fetching HTTP status code:[Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatus]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: RESTResponse.prototype.status :: line 643"  data: no] Stack trace: RESTResponse.prototype.status()@resource://services-common/rest.js:643 < _onComplete()@resource://services-common/hawkclient.js:218 < onComplete()@resource://services-common/hawkclient.js:275 < onStopRequest()@resource://services-common/rest.js:467 < waitForSyncCallback()@resource://services-common/async.js:102 < makeSpinningCallback/callback.wait()@resource://services-common/async.js:156 < onNotify()@resource://gre/modules/services-sync/service.js:1009 < WrappedNotify()@resource://services-sync/util.js:148 < WrappedLock()@resource://services-sync/util.js:103 < WrappedCatch()@resource://services-sync/util.js:77 < login()@resource://gre/modules/services-sync/service.js:1031 < sync/<()@resource://gre/modules/services-sync/service.js:1283 < WrappedCatch()@resource://services-sync/util.js:77 < sync()@resource://gre/modules/services-sync/service.js:1279
1566306746113	Sync.RESTResponse	DEBUG	Caught exception fetching HTTP status text:[Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatusText]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: RESTResponse.prototype.statusText :: line 659"  data: no] Stack trace: RESTResponse.prototype.statusText()@resource://services-common/rest.js:659 < _onComplete()@resource://services-common/hawkclient.js:221 < onComplete()@resource://services-common/hawkclient.js:275 < onStopRequest()@resource://services-common/rest.js:467 < waitForSyncCallback()@resource://services-common/async.js:102 < makeSpinningCallback/callback.wait()@resource://services-common/async.js:156 < onNotify()@resource://gre/modules/services-sync/service.js:1009 < WrappedNotify()@resource://services-sync/util.js:148 < WrappedLock()@resource://services-sync/util.js:103 < WrappedCatch()@resource://services-sync/util.js:77 < login()@resource://gre/modules/services-sync/service.js:1031 < sync/<()@resource://gre/modules/services-sync/service.js:1283 < WrappedCatch()@resource://services-sync/util.js:77 < sync()@resource://gre/modules/services-sync/service.js:1279
1566306746114	Hawk	DEBUG	(Response) /recovery_email/status: code: null - Status text: null
1566306746114	Sync.RESTResponse	TRACE	Processing response headers.
1566306746114	Sync.RESTResponse	DEBUG	Caught exception processing response headers:[Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.visitResponseHeaders]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: RESTResponse.prototype.headers :: line 693"  data: no] Stack trace: RESTResponse.prototype.headers()@resource://services-common/rest.js:693 < this.HawkClient.prototype._maybeNotifyBackoff()@resource://services-common/hawkclient.js:315 < _onComplete()@resource://services-common/hawkclient.js:228 < onComplete()@resource://services-common/hawkclient.js:275 < onStopRequest()@resource://services-common/rest.js:467 < waitForSyncCallback()@resource://services-common/async.js:102 < makeSpinningCallback/callback.wait()@resource://services-common/async.js:156 < onNotify()@resource://gre/modules/services-sync/service.js:1009 < WrappedNotify()@resource://services-sync/util.js:148 < WrappedLock()@resource://services-sync/util.js:103 < WrappedCatch()@resource://services-sync/util.js:77 < login()@resource://gre/modules/services-sync/service.js:1031 < sync/<()@resource://gre/modules/services-sync/service.js:1283 < WrappedCatch()@resource://services-sync/util.js:77 < sync()@resource://gre/modules/services-sync/service.js:1279
1566306746115	Sync.RESTResponse	TRACE	Processing response headers.
1566306746115	Sync.RESTResponse	DEBUG	Caught exception processing response headers:[Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.visitResponseHeaders]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: RESTResponse.prototype.headers :: line 693"  data: no] Stack trace: RESTResponse.prototype.headers()@resource://services-common/rest.js:693 < this.HawkClient.prototype._maybeNotifyBackoff()@resource://services-common/hawkclient.js:315 < _onComplete()@resource://services-common/hawkclient.js:229 < onComplete()@resource://services-common/hawkclient.js:275 < onStopRequest()@resource://services-common/rest.js:467 < waitForSyncCallback()@resource://services-common/async.js:102 < makeSpinningCallback/callback.wait()@resource://services-common/async.js:156 < onNotify()@resource://gre/modules/services-sync/service.js:1009 < WrappedNotify()@resource://services-sync/util.js:148 < WrappedLock()@resource://services-sync/util.js:103 < WrappedCatch()@resource://services-sync/util.js:77 < login()@resource://gre/modules/services-sync/service.js:1031 < sync/<()@resource://gre/modules/services-sync/service.js:1283 < WrappedCatch()@resource://services-sync/util.js:77 < sync()@resource://gre/modules/services-sync/service.js:1279
1566306746116	Sync.RESTResponse	DEBUG	Caught exception fetching HTTP status text:[Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatusText]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: RESTResponse.prototype.statusText :: line 659"  data: no] Stack trace: RESTResponse.prototype.statusText()@resource://services-common/rest.js:659 < this.HawkClient.prototype._constructError()@resource://services-common/hawkclient.js:121 < _onComplete()@resource://services-common/hawkclient.js:234 < onComplete()@resource://services-common/hawkclient.js:275 < onStopRequest()@resource://services-common/rest.js:467 < waitForSyncCallback()@resource://services-common/async.js:102 < makeSpinningCallback/callback.wait()@resource://services-common/async.js:156 < onNotify()@resource://gre/modules/services-sync/service.js:1009 < WrappedNotify()@resource://services-sync/util.js:148 < WrappedLock()@resource://services-sync/util.js:103 < WrappedCatch()@resource://services-sync/util.js:77 < login()@resource://gre/modules/services-sync/service.js:1031 < sync/<()@resource://gre/modules/services-sync/service.js:1283 < WrappedCatch()@resource://services-sync/util.js:77 < sync()@resource://gre/modules/services-sync/service.js:1279
1566306746117	Sync.RESTResponse	DEBUG	Caught exception fetching HTTP status code:[Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatus]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: RESTResponse.prototype.status :: line 643"  data: no] Stack trace: RESTResponse.prototype.status()@resource://services-common/rest.js:643 < this.HawkClient.prototype._constructError()@resource://services-common/hawkclient.js:122 < _onComplete()@resource://services-common/hawkclient.js:234 < onComplete()@resource://services-common/hawkclient.js:275 < onStopRequest()@resource://services-common/rest.js:467 < waitForSyncCallback()@resource://services-common/async.js:102 < makeSpinningCallback/callback.wait()@resource://services-common/async.js:156 < onNotify()@resource://gre/modules/services-sync/service.js:1009 < WrappedNotify()@resource://services-sync/util.js:148 < WrappedLock()@resource://services-sync/util.js:103 < WrappedCatch()@resource://services-sync/util.js:77 < login()@resource://gre/modules/services-sync/service.js:1031 < sync/<()@resource://gre/modules/services-sync/service.js:1283 < WrappedCatch()@resource://services-sync/util.js:77 < sync()@resource://gre/modules/services-sync/service.js:1279
1566306746118	Sync.RESTResponse	DEBUG	Caught exception fetching HTTP status code:[Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatus]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: RESTResponse.prototype.status :: line 643"  data: no] Stack trace: RESTResponse.prototype.status()@resource://services-common/rest.js:643 < this.HawkClient.prototype._constructError()@resource://services-common/hawkclient.js:123 < _onComplete()@resource://services-common/hawkclient.js:234 < onComplete()@resource://services-common/hawkclient.js:275 < onStopRequest()@resource://services-common/rest.js:467 < waitForSyncCallback()@resource://services-common/async.js:102 < makeSpinningCallback/callback.wait()@resource://services-common/async.js:156 < onNotify()@resource://gre/modules/services-sync/service.js:1009 < WrappedNotify()@resource://services-sync/util.js:148 < WrappedLock()@resource://services-sync/util.js:103 < WrappedCatch()@resource://services-sync/util.js:77 < login()@resource://gre/modules/services-sync/service.js:1031 < sync/<()@resource://gre/modules/services-sync/service.js:1283 < WrappedCatch()@resource://services-sync/util.js:77 < sync()@resource://gre/modules/services-sync/service.js:1279
1566306746119	Sync.RESTResponse	TRACE	Processing response headers.
1566306746119	Sync.RESTResponse	DEBUG	Caught exception processing response headers:[Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.visitResponseHeaders]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: RESTResponse.prototype.headers :: line 693"  data: no] Stack trace: RESTResponse.prototype.headers()@resource://services-common/rest.js:693 < this.HawkClient.prototype._constructError()@resource://services-common/hawkclient.js:128 < _onComplete()@resource://services-common/hawkclient.js:234 < onComplete()@resource://services-common/hawkclient.js:275 < onStopRequest()@resource://services-common/rest.js:467 < waitForSyncCallback()@resource://services-common/async.js:102 < makeSpinningCallback/callback.wait()@resource://services-common/async.js:156 < onNotify()@resource://gre/modules/services-sync/service.js:1009 < WrappedNotify()@resource://services-sync/util.js:148 < WrappedLock()@resource://services-sync/util.js:103 < WrappedCatch()@resource://services-sync/util.js:77 < login()@resource://gre/modules/services-sync/service.js:1031 < sync/<()@resource://gre/modules/services-sync/service.js:1283 < WrappedCatch()@resource://services-sync/util.js:77 < sync()@resource://gre/modules/services-sync/service.js:1279
1566306746120	FirefoxAccounts	ERROR	error GETing /recovery_email/status: {"error":{},"errorString":"[Exception... \"NS_ERROR_UNKNOWN_HOST\"  nsresult: \"0x804b001e (NS_ERROR_UNKNOWN_HOST)\"  location: \"JS frame :: resource://services-common/rest.js :: onStopRequest :: line 465\"  data: no]","message":null,"code":null,"errno":null}
1566306746121	FirefoxAccounts	DEBUG	polling with timeout = 5000
1566306751122	FirefoxAccounts	DEBUG	entering pollEmailStatus: timer
1566306751140	Hawk	WARN	hawk request error: [Exception... "NS_ERROR_UNKNOWN_HOST"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "JS frame :: resource://services-common/rest.js :: onStopRequest :: line 465"  data: no] Stack trace: onStopRequest()@resource://services-common/rest.js:465 < waitForSyncCallback()@resource://services-common/async.js:102 < makeSpinningCallback/callback.wait()@resource://services-common/async.js:156 < onNotify()@resource://gre/modules/services-sync/service.js:1009 < WrappedNotify()@resource://services-sync/util.js:148 < WrappedLock()@resource://services-sync/util.js:103 < WrappedCatch()@resource://services-sync/util.js:77 < login()@resource://gre/modules/services-sync/service.js:1031 < sync/<()@resource://gre/modules/services-sync/service.js:1283 < WrappedCatch()@resource://services-sync/util.js:77 < sync()@resource://gre/modules/services-sync/service.js:1279
1566306751142	Sync.RESTResponse	DEBUG	Caught exception fetching HTTP status code:[Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatus]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: RESTResponse.prototype.status :: line 643"  data: no] Stack trace: RESTResponse.prototype.status()@resource://services-common/rest.js:643 < _onComplete()@resource://services-common/hawkclient.js:218 < onComplete()@resource://services-common/hawkclient.js:275 < onStopRequest()@resource://services-common/rest.js:467 < waitForSyncCallback()@resource://services-common/async.js:102 < makeSpinningCallback/callback.wait()@resource://services-common/async.js:156 < onNotify()@resource://gre/modules/services-sync/service.js:1009 < WrappedNotify()@resource://services-sync/util.js:148 < WrappedLock()@resource://services-sync/util.js:103 < WrappedCatch()@resource://services-sync/util.js:77 < login()@resource://gre/modules/services-sync/service.js:1031 < sync/<()@resource://gre/modules/services-sync/service.js:1283 < WrappedCatch()@resource://services-sync/util.js:77 < sync()@resource://gre/modules/services-sync/service.js:1279
1566306751143	Sync.RESTResponse	DEBUG	Caught exception fetching HTTP status text:[Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatusText]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: RESTResponse.prototype.statusText :: line 659"  data: no] Stack trace: RESTResponse.prototype.statusText()@resource://services-common/rest.js:659 < _onComplete()@resource://services-common/hawkclient.js:221 < onComplete()@resource://services-common/hawkclient.js:275 < onStopRequest()@resource://services-common/rest.js:467 < waitForSyncCallback()@resource://services-common/async.js:102 < makeSpinningCallback/callback.wait()@resource://services-common/async.js:156 < onNotify()@resource://gre/modules/services-sync/service.js:1009 < WrappedNotify()@resource://services-sync/util.js:148 < WrappedLock()@resource://services-sync/util.js:103 < WrappedCatch()@resource://services-sync/util.js:77 < login()@resource://gre/modules/services-sync/service.js:1031 < sync/<()@resource://gre/modules/services-sync/service.js:1283 < WrappedCatch()@resource://services-sync/util.js:77 < sync()@resource://gre/modules/services-sync/service.js:1279
1566306751143	Hawk	DEBUG	(Response) /recovery_email/status: code: null - Status text: null
1566306751143	Sync.RESTResponse	TRACE	Processing response headers.
1566306751144	Sync.RESTResponse	DEBUG	Caught exception processing response headers:[Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.visitResponseHeaders]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: RESTResponse.prototype.headers :: line 693"  data: no] Stack trace: RESTResponse.prototype.headers()@resource://services-common/rest.js:693 < this.HawkClient.prototype._maybeNotifyBackoff()@resource://services-common/hawkclient.js:315 < _onComplete()@resource://services-common/hawkclient.js:228 < onComplete()@resource://services-common/hawkclient.js:275 < onStopRequest()@resource://services-common/rest.js:467 < waitForSyncCallback()@resource://services-common/async.js:102 < makeSpinningCallback/callback.wait()@resource://services-common/async.js:156 < onNotify()@resource://gre/modules/services-sync/service.js:1009 < WrappedNotify()@resource://services-sync/util.js:148 < WrappedLock()@resource://services-sync/util.js:103 < WrappedCatch()@resource://services-sync/util.js:77 < login()@resource://gre/modules/services-sync/service.js:1031 < sync/<()@resource://gre/modules/services-sync/service.js:1283 < WrappedCatch()@resource://services-sync/util.js:77 < sync()@resource://gre/modules/services-sync/service.js:1279
1566306751144	Sync.RESTResponse	TRACE	Processing response headers.
1566306751145	Sync.RESTResponse	DEBUG	Caught exception processing response headers:[Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.visitResponseHeaders]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: RESTResponse.prototype.headers :: line 693"  data: no] Stack trace: RESTResponse.prototype.headers()@resource://services-common/rest.js:693 < this.HawkClient.prototype._maybeNotifyBackoff()@resource://services-common/hawkclient.js:315 < _onComplete()@resource://services-common/hawkclient.js:229 < onComplete()@resource://services-common/hawkclient.js:275 < onStopRequest()@resource://services-common/rest.js:467 < waitForSyncCallback()@resource://services-common/async.js:102 < makeSpinningCallback/callback.wait()@resource://services-common/async.js:156 < onNotify()@resource://gre/modules/services-sync/service.js:1009 < WrappedNotify()@resource://services-sync/util.js:148 < WrappedLock()@resource://services-sync/util.js:103 < WrappedCatch()@resource://services-sync/util.js:77 < login()@resource://gre/modules/services-sync/service.js:1031 < sync/<()@resource://gre/modules/services-sync/service.js:1283 < WrappedCatch()@resource://services-sync/util.js:77 < sync()@resource://gre/modules/services-sync/service.js:1279
1566306751146	Sync.RESTResponse	DEBUG	Caught exception fetching HTTP status text:[Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatusText]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: RESTResponse.prototype.statusText :: line 659"  data: no] Stack trace: RESTResponse.prototype.statusText()@resource://services-common/rest.js:659 < this.HawkClient.prototype._constructError()@resource://services-common/hawkclient.js:121 < _onComplete()@resource://services-common/hawkclient.js:234 < onComplete()@resource://services-common/hawkclient.js:275 < onStopRequest()@resource://services-common/rest.js:467 < waitForSyncCallback()@resource://services-common/async.js:102 < makeSpinningCallback/callback.wait()@resource://services-common/async.js:156 < onNotify()@resource://gre/modules/services-sync/service.js:1009 < WrappedNotify()@resource://services-sync/util.js:148 < WrappedLock()@resource://services-sync/util.js:103 < WrappedCatch()@resource://services-sync/util.js:77 < login()@resource://gre/modules/services-sync/service.js:1031 < sync/<()@resource://gre/modules/services-sync/service.js:1283 < WrappedCatch()@resource://services-sync/util.js:77 < sync()@resource://gre/modules/services-sync/service.js:1279
1566306751147	Sync.RESTResponse	DEBUG	Caught exception fetching HTTP status code:[Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatus]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: RESTResponse.prototype.status :: line 643"  data: no] Stack trace: RESTResponse.prototype.status()@resource://services-common/rest.js:643 < this.HawkClient.prototype._constructError()@resource://services-common/hawkclient.js:122 < _onComplete()@resource://services-common/hawkclient.js:234 < onComplete()@resource://services-common/hawkclient.js:275 < onStopRequest()@resource://services-common/rest.js:467 < waitForSyncCallback()@resource://services-common/async.js:102 < makeSpinningCallback/callback.wait()@resource://services-common/async.js:156 < onNotify()@resource://gre/modules/services-sync/service.js:1009 < WrappedNotify()@resource://services-sync/util.js:148 < WrappedLock()@resource://services-sync/util.js:103 < WrappedCatch()@resource://services-sync/util.js:77 < login()@resource://gre/modules/services-sync/service.js:1031 < sync/<()@resource://gre/modules/services-sync/service.js:1283 < WrappedCatch()@resource://services-sync/util.js:77 < sync()@resource://gre/modules/services-sync/service.js:1279
1566306751148	Sync.RESTResponse	DEBUG	Caught exception fetching HTTP status code:[Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatus]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: RESTResponse.prototype.status :: line 643"  data: no] Stack trace: RESTResponse.prototype.status()@resource://services-common/rest.js:643 < this.HawkClient.prototype._constructError()@resource://services-common/hawkclient.js:123 < _onComplete()@resource://services-common/hawkclient.js:234 < onComplete()@resource://services-common/hawkclient.js:275 < onStopRequest()@resource://services-common/rest.js:467 < waitForSyncCallback()@resource://services-common/async.js:102 < makeSpinningCallback/callback.wait()@resource://services-common/async.js:156 < onNotify()@resource://gre/modules/services-sync/service.js:1009 < WrappedNotify()@resource://services-sync/util.js:148 < WrappedLock()@resource://services-sync/util.js:103 < WrappedCatch()@resource://services-sync/util.js:77 < login()@resource://gre/modules/services-sync/service.js:1031 < sync/<()@resource://gre/modules/services-sync/service.js:1283 < WrappedCatch()@resource://services-sync/util.js:77 < sync()@resource://gre/modules/services-sync/service.js:1279
1566306751149	Sync.RESTResponse	TRACE	Processing response headers.
1566306751149	Sync.RESTResponse	DEBUG	Caught exception processing response headers:[Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.visitResponseHeaders]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: RESTResponse.prototype.headers :: line 693"  data: no] Stack trace: RESTResponse.prototype.headers()@resource://services-common/rest.js:693 < this.HawkClient.prototype._constructError()@resource://services-common/hawkclient.js:128 < _onComplete()@resource://services-common/hawkclient.js:234 < onComplete()@resource://services-common/hawkclient.js:275 < onStopRequest()@resource://services-common/rest.js:467 < waitForSyncCallback()@resource://services-common/async.js:102 < makeSpinningCallback/callback.wait()@resource://services-common/async.js:156 < onNotify()@resource://gre/modules/services-sync/service.js:1009 < WrappedNotify()@resource://services-sync/util.js:148 < WrappedLock()@resource://services-sync/util.js:103 < WrappedCatch()@resource://services-sync/util.js:77 < login()@resource://gre/modules/services-sync/service.js:1031 < sync/<()@resource://gre/modules/services-sync/service.js:1283 < WrappedCatch()@resource://services-sync/util.js:77 < sync()@resource://gre/modules/services-sync/service.js:1279
1566306751150	FirefoxAccounts	ERROR	error GETing /recovery_email/status: {"error":{},"errorString":"[Exception... \"NS_ERROR_UNKNOWN_HOST\"  nsresult: \"0x804b001e (NS_ERROR_UNKNOWN_HOST)\"  location: \"JS frame :: resource://services-common/rest.js :: onStopRequest :: line 465\"  data: no]","message":null,"code":null,"errno":null}
1566306751151	FirefoxAccounts	DEBUG	polling with timeout = 5000
1566306756152	FirefoxAccounts	DEBUG	entering pollEmailStatus: timer
1566306756168	Hawk	WARN	hawk request error: [Exception... "NS_ERROR_UNKNOWN_HOST"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "JS frame :: resource://services-common/rest.js :: onStopRequest :: line 465"  data: no] Stack trace: onStopRequest()@resource://services-common/rest.js:465 < waitForSyncCallback()@resource://services-common/async.js:102 < makeSpinningCallback/callback.wait()@resource://services-common/async.js:156 < onNotify()@resource://gre/modules/services-sync/service.js:1009 < WrappedNotify()@resource://services-sync/util.js:148 < WrappedLock()@resource://services-sync/util.js:103 < WrappedCatch()@resource://services-sync/util.js:77 < login()@resource://gre/modules/services-sync/service.js:1031 < sync/<()@resource://gre/modules/services-sync/service.js:1283 < WrappedCatch()@resource://services-sync/util.js:77 < sync()@resource://gre/modules/services-sync/service.js:1279
1566306756170	Sync.RESTResponse	DEBUG	Caught exception fetching HTTP status code:[Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatus]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: RESTResponse.prototype.status :: line 643"  data: no] Stack trace: RESTResponse.prototype.status()@resource://services-common/rest.js:643 < _onComplete()@resource://services-common/hawkclient.js:218 < onComplete()@resource://services-common/hawkclient.js:275 < onStopRequest()@resource://services-common/rest.js:467 < waitForSyncCallback()@resource://services-common/async.js:102 < makeSpinningCallback/callback.wait()@resource://services-common/async.js:156 < onNotify()@resource://gre/modules/services-sync/service.js:1009 < WrappedNotify()@resource://services-sync/util.js:148 < WrappedLock()@resource://services-sync/util.js:103 < WrappedCatch()@resource://services-sync/util.js:77 < login()@resource://gre/modules/services-sync/service.js:1031 < sync/<()@resource://gre/modules/services-sync/service.js:1283 < WrappedCatch()@resource://services-sync/util.js:77 < sync()@resource://gre/modules/services-sync/service.js:1279
1566306756170	Sync.RESTResponse	DEBUG	Caught exception fetching HTTP status text:[Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatusText]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: RESTResponse.prototype.statusText :: line 659"  data: no] Stack trace: RESTResponse.prototype.statusText()@resource://services-common/rest.js:659 < _onComplete()@resource://services-common/hawkclient.js:221 < onComplete()@resource://services-common/hawkclient.js:275 < onStopRequest()@resource://services-common/rest.js:467 < waitForSyncCallback()@resource://services-common/async.js:102 < makeSpinningCallback/callback.wait()@resource://services-common/async.js:156 < onNotify()@resource://gre/modules/services-sync/service.js:1009 < WrappedNotify()@resource://services-sync/util.js:148 < WrappedLock()@resource://services-sync/util.js:103 < WrappedCatch()@resource://services-sync/util.js:77 < login()@resource://gre/modules/services-sync/service.js:1031 < sync/<()@resource://gre/modules/services-sync/service.js:1283 < WrappedCatch()@resource://services-sync/util.js:77 < sync()@resource://gre/modules/services-sync/service.js:1279
1566306756171	Hawk	DEBUG	(Response) /recovery_email/status: code: null - Status text: null
1566306756171	Sync.RESTResponse	TRACE	Processing response headers.
1566306756172	Sync.RESTResponse	DEBUG	Caught exception processing response headers:[Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.visitResponseHeaders]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: RESTResponse.prototype.headers :: line 693"  data: no] Stack trace: RESTResponse.prototype.headers()@resource://services-common/rest.js:693 < this.HawkClient.prototype._maybeNotifyBackoff()@resource://services-common/hawkclient.js:315 < _onComplete()@resource://services-common/hawkclient.js:228 < onComplete()@resource://services-common/hawkclient.js:275 < onStopRequest()@resource://services-common/rest.js:467 < waitForSyncCallback()@resource://services-common/async.js:102 < makeSpinningCallback/callback.wait()@resource://services-common/async.js:156 < onNotify()@resource://gre/modules/services-sync/service.js:1009 < WrappedNotify()@resource://services-sync/util.js:148 < WrappedLock()@resource://services-sync/util.js:103 < WrappedCatch()@resource://services-sync/util.js:77 < login()@resource://gre/modules/services-sync/service.js:1031 < sync/<()@resource://gre/modules/services-sync/service.js:1283 < WrappedCatch()@resource://services-sync/util.js:77 < sync()@resource://gre/modules/services-sync/service.js:1279
1566306756172	Sync.RESTResponse	TRACE	Processing response headers.
1566306756173	Sync.RESTResponse	DEBUG	Caught exception processing response headers:[Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.visitResponseHeaders]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: RESTResponse.prototype.headers :: line 693"  data: no] Stack trace: RESTResponse.prototype.headers()@resource://services-common/rest.js:693 < this.HawkClient.prototype._maybeNotifyBackoff()@resource://services-common/hawkclient.js:315 < _onComplete()@resource://services-common/hawkclient.js:229 < onComplete()@resource://services-common/hawkclient.js:275 < onStopRequest()@resource://services-common/rest.js:467 < waitForSyncCallback()@resource://services-common/async.js:102 < makeSpinningCallback/callback.wait()@resource://services-common/async.js:156 < onNotify()@resource://gre/modules/services-sync/service.js:1009 < WrappedNotify()@resource://services-sync/util.js:148 < WrappedLock()@resource://services-sync/util.js:103 < WrappedCatch()@resource://services-sync/util.js:77 < login()@resource://gre/modules/services-sync/service.js:1031 < sync/<()@resource://gre/modules/services-sync/service.js:1283 < WrappedCatch()@resource://services-sync/util.js:77 < sync()@resource://gre/modules/services-sync/service.js:1279
1566306756174	Sync.RESTResponse	DEBUG	Caught exception fetching HTTP status text:[Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatusText]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: RESTResponse.prototype.statusText :: line 659"  data: no] Stack trace: RESTResponse.prototype.statusText()@resource://services-common/rest.js:659 < this.HawkClient.prototype._constructError()@resource://services-common/hawkclient.js:121 < _onComplete()@resource://services-common/hawkclient.js:234 < onComplete()@resource://services-common/hawkclient.js:275 < onStopRequest()@resource://services-common/rest.js:467 < waitForSyncCallback()@resource://services-common/async.js:102 < makeSpinningCallback/callback.wait()@resource://services-common/async.js:156 < onNotify()@resource://gre/modules/services-sync/service.js:1009 < WrappedNotify()@resource://services-sync/util.js:148 < WrappedLock()@resource://services-sync/util.js:103 < WrappedCatch()@resource://services-sync/util.js:77 < login()@resource://gre/modules/services-sync/service.js:1031 < sync/<()@resource://gre/modules/services-sync/service.js:1283 < WrappedCatch()@resource://services-sync/util.js:77 < sync()@resource://gre/modules/services-sync/service.js:1279
1566306756174	Sync.RESTResponse	DEBUG	Caught exception fetching HTTP status code:[Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatus]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: RESTResponse.prototype.status :: line 643"  data: no] Stack trace: RESTResponse.prototype.status()@resource://services-common/rest.js:643 < this.HawkClient.prototype._constructError()@resource://services-common/hawkclient.js:122 < _onComplete()@resource://services-common/hawkclient.js:234 < onComplete()@resource://services-common/hawkclient.js:275 < onStopRequest()@resource://services-common/rest.js:467 < waitForSyncCallback()@resource://services-common/async.js:102 < makeSpinningCallback/callback.wait()@resource://services-common/async.js:156 < onNotify()@resource://gre/modules/services-sync/service.js:1009 < WrappedNotify()@resource://services-sync/util.js:148 < WrappedLock()@resource://services-sync/util.js:103 < WrappedCatch()@resource://services-sync/util.js:77 < login()@resource://gre/modules/services-sync/service.js:1031 < sync/<()@resource://gre/modules/services-sync/service.js:1283 < WrappedCatch()@resource://services-sync/util.js:77 < sync()@resource://gre/modules/services-sync/service.js:1279
1566306756175	Sync.RESTResponse	DEBUG	Caught exception fetching HTTP status code:[Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatus]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: RESTResponse.prototype.status :: line 643"  data: no] Stack trace: RESTResponse.prototype.status()@resource://services-common/rest.js:643 < this.HawkClient.prototype._constructError()@resource://services-common/hawkclient.js:123 < _onComplete()@resource://services-common/hawkclient.js:234 < onComplete()@resource://services-common/hawkclient.js:275 < onStopRequest()@resource://services-common/rest.js:467 < waitForSyncCallback()@resource://services-common/async.js:102 < makeSpinningCallback/callback.wait()@resource://services-common/async.js:156 < onNotify()@resource://gre/modules/services-sync/service.js:1009 < WrappedNotify()@resource://services-sync/util.js:148 < WrappedLock()@resource://services-sync/util.js:103 < WrappedCatch()@resource://services-sync/util.js:77 < login()@resource://gre/modules/services-sync/service.js:1031 < sync/<()@resource://gre/modules/services-sync/service.js:1283 < WrappedCatch()@resource://services-sync/util.js:77 < sync()@resource://gre/modules/services-sync/service.js:1279
1566306756175	Sync.RESTResponse	TRACE	Processing response headers.
1566306756176	Sync.RESTResponse	DEBUG	Caught exception processing response headers:[Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.visitResponseHeaders]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: RESTResponse.prototype.headers :: line 693"  data: no] Stack trace: RESTResponse.prototype.headers()@resource://services-common/rest.js:693 < this.HawkClient.prototype._constructError()@resource://services-common/hawkclient.js:128 < _onComplete()@resource://services-common/hawkclient.js:234 < onComplete()@resource://services-common/hawkclient.js:275 < onStopRequest()@resource://services-common/rest.js:467 < waitForSyncCallback()@resource://services-common/async.js:102 < makeSpinningCallback/callback.wait()@resource://services-common/async.js:156 < onNotify()@resource://gre/modules/services-sync/service.js:1009 < WrappedNotify()@resource://services-sync/util.js:148 < WrappedLock()@resource://services-sync/util.js:103 < WrappedCatch()@resource://services-sync/util.js:77 < login()@resource://gre/modules/services-sync/service.js:1031 < sync/<()@resource://gre/modules/services-sync/service.js:1283 < WrappedCatch()@resource://services-sync/util.js:77 < sync()@resource://gre/modules/services-sync/service.js:1279
1566306756177	FirefoxAccounts	ERROR	error GETing /recovery_email/status: {"error":{},"errorString":"[Exception... \"NS_ERROR_UNKNOWN_HOST\"  nsresult: \"0x804b001e (NS_ERROR_UNKNOWN_HOST)\"  location: \"JS frame :: resource://services-common/rest.js :: onStopRequest :: line 465\"  data: no]","message":null,"code":null,"errno":null}
1566306756178	FirefoxAccounts	DEBUG	polling with timeout = 5000
1566306761179	FirefoxAccounts	DEBUG	entering pollEmailStatus: timer
1566306761192	Hawk	WARN	hawk request error: [Exception... "NS_ERROR_UNKNOWN_HOST"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "JS frame :: resource://services-common/rest.js :: onStopRequest :: line 465"  data: no] Stack trace: onStopRequest()@resource://services-common/rest.js:465 < waitForSyncCallback()@resource://services-common/async.js:102 < makeSpinningCallback/callback.wait()@resource://services-common/async.js:156 < onNotify()@resource://gre/modules/services-sync/service.js:1009 < WrappedNotify()@resource://services-sync/util.js:148 < WrappedLock()@resource://services-sync/util.js:103 < WrappedCatch()@resource://services-sync/util.js:77 < login()@resource://gre/modules/services-sync/service.js:1031 < sync/<()@resource://gre/modules/services-sync/service.js:1283 < WrappedCatch()@resource://services-sync/util.js:77 < sync()@resource://gre/modules/services-sync/service.js:1279
1566306761194	Sync.RESTResponse	DEBUG	Caught exception fetching HTTP status code:[Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatus]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: RESTResponse.prototype.status :: line 643"  data: no] Stack trace: RESTResponse.prototype.status()@resource://services-common/rest.js:643 < _onComplete()@resource://services-common/hawkclient.js:218 < onComplete()@resource://services-common/hawkclient.js:275 < onStopRequest()@resource://services-common/rest.js:467 < waitForSyncCallback()@resource://services-common/async.js:102 < makeSpinningCallback/callback.wait()@resource://services-common/async.js:156 < onNotify()@resource://gre/modules/services-sync/service.js:1009 < WrappedNotify()@resource://services-sync/util.js:148 < WrappedLock()@resource://services-sync/util.js:103 < WrappedCatch()@resource://services-sync/util.js:77 < login()@resource://gre/modules/services-sync/service.js:1031 < sync/<()@resource://gre/modules/services-sync/service.js:1283 < WrappedCatch()@resource://services-sync/util.js:77 < sync()@resource://gre/modules/services-sync/service.js:1279
1566306761194	Sync.RESTResponse	DEBUG	Caught exception fetching HTTP status text:[Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatusText]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: RESTResponse.prototype.statusText :: line 659"  data: no] Stack trace: RESTResponse.prototype.statusText()@resource://services-common/rest.js:659 < _onComplete()@resource://services-common/hawkclient.js:221 < onComplete()@resource://services-common/hawkclient.js:275 < onStopRequest()@resource://services-common/rest.js:467 < waitForSyncCallback()@resource://services-common/async.js:102 < makeSpinningCallback/callback.wait()@resource://services-common/async.js:156 < onNotify()@resource://gre/modules/services-sync/service.js:1009 < WrappedNotify()@resource://services-sync/util.js:148 < WrappedLock()@resource://services-sync/util.js:103 < WrappedCatch()@resource://services-sync/util.js:77 < login()@resource://gre/modules/services-sync/service.js:1031 < sync/<()@resource://gre/modules/services-sync/service.js:1283 < WrappedCatch()@resource://services-sync/util.js:77 < sync()@resource://gre/modules/services-sync/service.js:1279
1566306761195	Hawk	DEBUG	(Response) /recovery_email/status: code: null - Status text: null
1566306761195	Sync.RESTResponse	TRACE	Processing response headers.
1566306761196	Sync.RESTResponse	DEBUG	Caught exception processing response headers:[Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.visitResponseHeaders]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: RESTResponse.prototype.headers :: line 693"  data: no] Stack trace: RESTResponse.prototype.headers()@resource://services-common/rest.js:693 < this.HawkClient.prototype._maybeNotifyBackoff()@resource://services-common/hawkclient.js:315 < _onComplete()@resource://services-common/hawkclient.js:228 < onComplete()@resource://services-common/hawkclient.js:275 < onStopRequest()@resource://services-common/rest.js:467 < waitForSyncCallback()@resource://services-common/async.js:102 < makeSpinningCallback/callback.wait()@resource://services-common/async.js:156 < onNotify()@resource://gre/modules/services-sync/service.js:1009 < WrappedNotify()@resource://services-sync/util.js:148 < WrappedLock()@resource://services-sync/util.js:103 < WrappedCatch()@resource://services-sync/util.js:77 < login()@resource://gre/modules/services-sync/service.js:1031 < sync/<()@resource://gre/modules/services-sync/service.js:1283 < WrappedCatch()@resource://services-sync/util.js:77 < sync()@resource://gre/modules/services-sync/service.js:1279
1566306761196	Sync.RESTResponse	TRACE	Processing response headers.
1566306761197	Sync.RESTResponse	DEBUG	Caught exception processing response headers:[Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.visitResponseHeaders]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: RESTResponse.prototype.headers :: line 693"  data: no] Stack trace: RESTResponse.prototype.headers()@resource://services-common/rest.js:693 < this.HawkClient.prototype._maybeNotifyBackoff()@resource://services-common/hawkclient.js:315 < _onComplete()@resource://services-common/hawkclient.js:229 < onComplete()@resource://services-common/hawkclient.js:275 < onStopRequest()@resource://services-common/rest.js:467 < waitForSyncCallback()@resource://services-common/async.js:102 < makeSpinningCallback/callback.wait()@resource://services-common/async.js:156 < onNotify()@resource://gre/modules/services-sync/service.js:1009 < WrappedNotify()@resource://services-sync/util.js:148 < WrappedLock()@resource://services-sync/util.js:103 < WrappedCatch()@resource://services-sync/util.js:77 < login()@resource://gre/modules/services-sync/service.js:1031 < sync/<()@resource://gre/modules/services-sync/service.js:1283 < WrappedCatch()@resource://services-sync/util.js:77 < sync()@resource://gre/modules/services-sync/service.js:1279
1566306761198	Sync.RESTResponse	DEBUG	Caught exception fetching HTTP status text:[Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatusText]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: RESTResponse.prototype.statusText :: line 659"  data: no] Stack trace: RESTResponse.prototype.statusText()@resource://services-common/rest.js:659 < this.HawkClient.prototype._constructError()@resource://services-common/hawkclient.js:121 < _onComplete()@resource://services-common/hawkclient.js:234 < onComplete()@resource://services-common/hawkclient.js:275 < onStopRequest()@resource://services-common/rest.js:467 < waitForSyncCallback()@resource://services-common/async.js:102 < makeSpinningCallback/callback.wait()@resource://services-common/async.js:156 < onNotify()@resource://gre/modules/services-sync/service.js:1009 < WrappedNotify()@resource://services-sync/util.js:148 < WrappedLock()@resource://services-sync/util.js:103 < WrappedCatch()@resource://services-sync/util.js:77 < login()@resource://gre/modules/services-sync/service.js:1031 < sync/<()@resource://gre/modules/services-sync/service.js:1283 < WrappedCatch()@resource://services-sync/util.js:77 < sync()@resource://gre/modules/services-sync/service.js:1279
1566306761199	Sync.RESTResponse	DEBUG	Caught exception fetching HTTP status code:[Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatus]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: RESTResponse.prototype.status :: line 643"  data: no] Stack trace: RESTResponse.prototype.status()@resource://services-common/rest.js:643 < this.HawkClient.prototype._constructError()@resource://services-common/hawkclient.js:122 < _onComplete()@resource://services-common/hawkclient.js:234 < onComplete()@resource://services-common/hawkclient.js:275 < onStopRequest()@resource://services-common/rest.js:467 < waitForSyncCallback()@resource://services-common/async.js:102 < makeSpinningCallback/callback.wait()@resource://services-common/async.js:156 < onNotify()@resource://gre/modules/services-sync/service.js:1009 < WrappedNotify()@resource://services-sync/util.js:148 < WrappedLock()@resource://services-sync/util.js:103 < WrappedCatch()@resource://services-sync/util.js:77 < login()@resource://gre/modules/services-sync/service.js:1031 < sync/<()@resource://gre/modules/services-sync/service.js:1283 < WrappedCatch()@resource://services-sync/util.js:77 < sync()@resource://gre/modules/services-sync/service.js:1279
1566306761200	Sync.RESTResponse	DEBUG	Caught exception fetching HTTP status code:[Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatus]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: RESTResponse.prototype.status :: line 643"  data: no] Stack trace: RESTResponse.prototype.status()@resource://services-common/rest.js:643 < this.HawkClient.prototype._constructError()@resource://services-common/hawkclient.js:123 < _onComplete()@resource://services-common/hawkclient.js:234 < onComplete()@resource://services-common/hawkclient.js:275 < onStopRequest()@resource://services-common/rest.js:467 < waitForSyncCallback()@resource://services-common/async.js:102 < makeSpinningCallback/callback.wait()@resource://services-common/async.js:156 < onNotify()@resource://gre/modules/services-sync/service.js:1009 < WrappedNotify()@resource://services-sync/util.js:148 < WrappedLock()@resource://services-sync/util.js:103 < WrappedCatch()@resource://services-sync/util.js:77 < login()@resource://gre/modules/services-sync/service.js:1031 < sync/<()@resource://gre/modules/services-sync/service.js:1283 < WrappedCatch()@resource://services-sync/util.js:77 < sync()@resource://gre/modules/services-sync/service.js:1279
1566306761200	Sync.RESTResponse	TRACE	Processing response headers.
1566306761201	Sync.RESTResponse	DEBUG	Caught exception processing response headers:[Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.visitResponseHeaders]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: RESTResponse.prototype.headers :: line 693"  data: no] Stack trace: RESTResponse.prototype.headers()@resource://services-common/rest.js:693 < this.HawkClient.prototype._constructError()@resource://services-common/hawkclient.js:128 < _onComplete()@resource://services-common/hawkclient.js:234 < onComplete()@resource://services-common/hawkclient.js:275 < onStopRequest()@resource://services-common/rest.js:467 < waitForSyncCallback()@resource://services-common/async.js:102 < makeSpinningCallback/callback.wait()@resource://services-common/async.js:156 < onNotify()@resource://gre/modules/services-sync/service.js:1009 < WrappedNotify()@resource://services-sync/util.js:148 < WrappedLock()@resource://services-sync/util.js:103 < WrappedCatch()@resource://services-sync/util.js:77 < login()@resource://gre/modules/services-sync/service.js:1031 < sync/<()@resource://gre/modules/services-sync/service.js:1283 < WrappedCatch()@resource://services-sync/util.js:77 < sync()@resource://gre/modules/services-sync/service.js:1279
1566306761201	FirefoxAccounts	ERROR	error GETing /recovery_email/status: {"error":{},"errorString":"[Exception... \"NS_ERROR_UNKNOWN_HOST\"  nsresult: \"0x804b001e (NS_ERROR_UNKNOWN_HOST)\"  location: \"JS frame :: resource://services-common/rest.js :: onStopRequest :: line 465\"  data: no]","message":null,"code":null,"errno":null}
1566306761202	FirefoxAccounts	DEBUG	polling with timeout = 5000
1566306766203	FirefoxAccounts	DEBUG	entering pollEmailStatus: timer
1566306766220	Hawk	WARN	hawk request error: [Exception... "NS_ERROR_UNKNOWN_HOST"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "JS frame :: resource://services-common/rest.js :: onStopRequest :: line 465"  data: no] Stack trace: onStopRequest()@resource://services-common/rest.js:465 < waitForSyncCallback()@resource://services-common/async.js:102 < makeSpinningCallback/callback.wait()@resource://services-common/async.js:156 < onNotify()@resource://gre/modules/services-sync/service.js:1009 < WrappedNotify()@resource://services-sync/util.js:148 < WrappedLock()@resource://services-sync/util.js:103 < WrappedCatch()@resource://services-sync/util.js:77 < login()@resource://gre/modules/services-sync/service.js:1031 < sync/<()@resource://gre/modules/services-sync/service.js:1283 < WrappedCatch()@resource://services-sync/util.js:77 < sync()@resource://gre/modules/services-sync/service.js:1279
1566306766221	Sync.RESTResponse	DEBUG	Caught exception fetching HTTP status code:[Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatus]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: RESTResponse.prototype.status :: line 643"  data: no] Stack trace: RESTResponse.prototype.status()@resource://services-common/rest.js:643 < _onComplete()@resource://services-common/hawkclient.js:218 < onComplete()@resource://services-common/hawkclient.js:275 < onStopRequest()@resource://services-common/rest.js:467 < waitForSyncCallback()@resource://services-common/async.js:102 < makeSpinningCallback/callback.wait()@resource://services-common/async.js:156 < onNotify()@resource://gre/modules/services-sync/service.js:1009 < WrappedNotify()@resource://services-sync/util.js:148 < WrappedLock()@resource://services-sync/util.js:103 < WrappedCatch()@resource://services-sync/util.js:77 < login()@resource://gre/modules/services-sync/service.js:1031 < sync/<()@resource://gre/modules/services-sync/service.js:1283 < WrappedCatch()@resource://services-sync/util.js:77 < sync()@resource://gre/modules/services-sync/service.js:1279
1566306766222	Sync.RESTResponse	DEBUG	Caught exception fetching HTTP status text:[Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatusText]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: RESTResponse.prototype.statusText :: line 659"  data: no] Stack trace: RESTResponse.prototype.statusText()@resource://services-common/rest.js:659 < _onComplete()@resource://services-common/hawkclient.js:221 < onComplete()@resource://services-common/hawkclient.js:275 < onStopRequest()@resource://services-common/rest.js:467 < waitForSyncCallback()@resource://services-common/async.js:102 < makeSpinningCallback/callback.wait()@resource://services-common/async.js:156 < onNotify()@resource://gre/modules/services-sync/service.js:1009 < WrappedNotify()@resource://services-sync/util.js:148 < WrappedLock()@resource://services-sync/util.js:103 < WrappedCatch()@resource://services-sync/util.js:77 < login()@resource://gre/modules/services-sync/service.js:1031 < sync/<()@resource://gre/modules/services-sync/service.js:1283 < WrappedCatch()@resource://services-sync/util.js:77 < sync()@resource://gre/modules/services-sync/service.js:1279
1566306766222	Hawk	DEBUG	(Response) /recovery_email/status: code: null - Status text: null
1566306766222	Sync.RESTResponse	TRACE	Processing response headers.
1566306766223	Sync.RESTResponse	DEBUG	Caught exception processing response headers:[Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.visitResponseHeaders]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: RESTResponse.prototype.headers :: line 693"  data: no] Stack trace: RESTResponse.prototype.headers()@resource://services-common/rest.js:693 < this.HawkClient.prototype._maybeNotifyBackoff()@resource://services-common/hawkclient.js:315 < _onComplete()@resource://services-common/hawkclient.js:228 < onComplete()@resource://services-common/hawkclient.js:275 < onStopRequest()@resource://services-common/rest.js:467 < waitForSyncCallback()@resource://services-common/async.js:102 < makeSpinningCallback/callback.wait()@resource://services-common/async.js:156 < onNotify()@resource://gre/modules/services-sync/service.js:1009 < WrappedNotify()@resource://services-sync/util.js:148 < WrappedLock()@resource://services-sync/util.js:103 < WrappedCatch()@resource://services-sync/util.js:77 < login()@resource://gre/modules/services-sync/service.js:1031 < sync/<()@resource://gre/modules/services-sync/service.js:1283 < WrappedCatch()@resource://services-sync/util.js:77 < sync()@resource://gre/modules/services-sync/service.js:1279
1566306766223	Sync.RESTResponse	TRACE	Processing response headers.
1566306766224	Sync.RESTResponse	DEBUG	Caught exception processing response headers:[Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.visitResponseHeaders]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: RESTResponse.prototype.headers :: line 693"  data: no] Stack trace: RESTResponse.prototype.headers()@resource://services-common/rest.js:693 < this.HawkClient.prototype._maybeNotifyBackoff()@resource://services-common/hawkclient.js:315 < _onComplete()@resource://services-common/hawkclient.js:229 < onComplete()@resource://services-common/hawkclient.js:275 < onStopRequest()@resource://services-common/rest.js:467 < waitForSyncCallback()@resource://services-common/async.js:102 < makeSpinningCallback/callback.wait()@resource://services-common/async.js:156 < onNotify()@resource://gre/modules/services-sync/service.js:1009 < WrappedNotify()@resource://services-sync/util.js:148 < WrappedLock()@resource://services-sync/util.js:103 < WrappedCatch()@resource://services-sync/util.js:77 < login()@resource://gre/modules/services-sync/service.js:1031 < sync/<()@resource://gre/modules/services-sync/service.js:1283 < WrappedCatch()@resource://services-sync/util.js:77 < sync()@resource://gre/modules/services-sync/service.js:1279
1566306766225	Sync.RESTResponse	DEBUG	Caught exception fetching HTTP status text:[Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatusText]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: RESTResponse.prototype.statusText :: line 659"  data: no] Stack trace: RESTResponse.prototype.statusText()@resource://services-common/rest.js:659 < this.HawkClient.prototype._constructError()@resource://services-common/hawkclient.js:121 < _onComplete()@resource://services-common/hawkclient.js:234 < onComplete()@resource://services-common/hawkclient.js:275 < onStopRequest()@resource://services-common/rest.js:467 < waitForSyncCallback()@resource://services-common/async.js:102 < makeSpinningCallback/callback.wait()@resource://services-common/async.js:156 < onNotify()@resource://gre/modules/services-sync/service.js:1009 < WrappedNotify()@resource://services-sync/util.js:148 < WrappedLock()@resource://services-sync/util.js:103 < WrappedCatch()@resource://services-sync/util.js:77 < login()@resource://gre/modules/services-sync/service.js:1031 < sync/<()@resource://gre/modules/services-sync/service.js:1283 < WrappedCatch()@resource://services-sync/util.js:77 < sync()@resource://gre/modules/services-sync/service.js:1279
1566306766226	Sync.RESTResponse	DEBUG	Caught exception fetching HTTP status code:[Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatus]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: RESTResponse.prototype.status :: line 643"  data: no] Stack trace: RESTResponse.prototype.status()@resource://services-common/rest.js:643 < this.HawkClient.prototype._constructError()@resource://services-common/hawkclient.js:122 < _onComplete()@resource://services-common/hawkclient.js:234 < onComplete()@resource://services-common/hawkclient.js:275 < onStopRequest()@resource://services-common/rest.js:467 < waitForSyncCallback()@resource://services-common/async.js:102 < makeSpinningCallback/callback.wait()@resource://services-common/async.js:156 < onNotify()@resource://gre/modules/services-sync/service.js:1009 < WrappedNotify()@resource://services-sync/util.js:148 < WrappedLock()@resource://services-sync/util.js:103 < WrappedCatch()@resource://services-sync/util.js:77 < login()@resource://gre/modules/services-sync/service.js:1031 < sync/<()@resource://gre/modules/services-sync/service.js:1283 < WrappedCatch()@resource://services-sync/util.js:77 < sync()@resource://gre/modules/services-sync/service.js:1279
1566306766227	Sync.RESTResponse	DEBUG	Caught exception fetching HTTP status code:[Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatus]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: RESTResponse.prototype.status :: line 643"  data: no] Stack trace: RESTResponse.prototype.status()@resource://services-common/rest.js:643 < this.HawkClient.prototype._constructError()@resource://services-common/hawkclient.js:123 < _onComplete()@resource://services-common/hawkclient.js:234 < onComplete()@resource://services-common/hawkclient.js:275 < onStopRequest()@resource://services-common/rest.js:467 < waitForSyncCallback()@resource://services-common/async.js:102 < makeSpinningCallback/callback.wait()@resource://services-common/async.js:156 < onNotify()@resource://gre/modules/services-sync/service.js:1009 < WrappedNotify()@resource://services-sync/util.js:148 < WrappedLock()@resource://services-sync/util.js:103 < WrappedCatch()@resource://services-sync/util.js:77 < login()@resource://gre/modules/services-sync/service.js:1031 < sync/<()@resource://gre/modules/services-sync/service.js:1283 < WrappedCatch()@resource://services-sync/util.js:77 < sync()@resource://gre/modules/services-sync/service.js:1279
1566306766227	Sync.RESTResponse	TRACE	Processing response headers.
1566306766228	Sync.RESTResponse	DEBUG	Caught exception processing response headers:[Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.visitResponseHeaders]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: RESTResponse.prototype.headers :: line 693"  data: no] Stack trace: RESTResponse.prototype.headers()@resource://services-common/rest.js:693 < this.HawkClient.prototype._constructError()@resource://services-common/hawkclient.js:128 < _onComplete()@resource://services-common/hawkclient.js:234 < onComplete()@resource://services-common/hawkclient.js:275 < onStopRequest()@resource://services-common/rest.js:467 < waitForSyncCallback()@resource://services-common/async.js:102 < makeSpinningCallback/callback.wait()@resource://services-common/async.js:156 < onNotify()@resource://gre/modules/services-sync/service.js:1009 < WrappedNotify()@resource://services-sync/util.js:148 < WrappedLock()@resource://services-sync/util.js:103 < WrappedCatch()@resource://services-sync/util.js:77 < login()@resource://gre/modules/services-sync/service.js:1031 < sync/<()@resource://gre/modules/services-sync/service.js:1283 < WrappedCatch()@resource://services-sync/util.js:77 < sync()@resource://gre/modules/services-sync/service.js:1279
1566306766229	FirefoxAccounts	ERROR	error GETing /recovery_email/status: {"error":{},"errorString":"[Exception... \"NS_ERROR_UNKNOWN_HOST\"  nsresult: \"0x804b001e (NS_ERROR_UNKNOWN_HOST)\"  location: \"JS frame :: resource://services-common/rest.js :: onStopRequest :: line 465\"  data: no]","message":null,"code":null,"errno":null}
1566306766230	FirefoxAccounts	DEBUG	polling with timeout = 5000
1566306771231	FirefoxAccounts	DEBUG	entering pollEmailStatus: timer
1566306771244	Hawk	WARN	hawk request error: [Exception... "NS_ERROR_UNKNOWN_HOST"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "JS frame :: resource://services-common/rest.js :: onStopRequest :: line 465"  data: no] Stack trace: onStopRequest()@resource://services-common/rest.js:465 < waitForSyncCallback()@resource://services-common/async.js:102 < makeSpinningCallback/callback.wait()@resource://services-common/async.js:156 < onNotify()@resource://gre/modules/services-sync/service.js:1009 < WrappedNotify()@resource://services-sync/util.js:148 < WrappedLock()@resource://services-sync/util.js:103 < WrappedCatch()@resource://services-sync/util.js:77 < login()@resource://gre/modules/services-sync/service.js:1031 < sync/<()@resource://gre/modules/services-sync/service.js:1283 < WrappedCatch()@resource://services-sync/util.js:77 < sync()@resource://gre/modules/services-sync/service.js:1279
1566306771246	Sync.RESTResponse	DEBUG	Caught exception fetching HTTP status code:[Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatus]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: RESTResponse.prototype.status :: line 643"  data: no] Stack trace: RESTResponse.prototype.status()@resource://services-common/rest.js:643 < _onComplete()@resource://services-common/hawkclient.js:218 < onComplete()@resource://services-common/hawkclient.js:275 < onStopRequest()@resource://services-common/rest.js:467 < waitForSyncCallback()@resource://services-common/async.js:102 < makeSpinningCallback/callback.wait()@resource://services-common/async.js:156 < onNotify()@resource://gre/modules/services-sync/service.js:1009 < WrappedNotify()@resource://services-sync/util.js:148 < WrappedLock()@resource://services-sync/util.js:103 < WrappedCatch()@resource://services-sync/util.js:77 < login()@resource://gre/modules/services-sync/service.js:1031 < sync/<()@resource://gre/modules/services-sync/service.js:1283 < WrappedCatch()@resource://services-sync/util.js:77 < sync()@resource://gre/modules/services-sync/service.js:1279
1566306771246	Sync.RESTResponse	DEBUG	Caught exception fetching HTTP status text:[Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatusText]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: RESTResponse.prototype.statusText :: line 659"  data: no] Stack trace: RESTResponse.prototype.statusText()@resource://services-common/rest.js:659 < _onComplete()@resource://services-common/hawkclient.js:221 < onComplete()@resource://services-common/hawkclient.js:275 < onStopRequest()@resource://services-common/rest.js:467 < waitForSyncCallback()@resource://services-common/async.js:102 < makeSpinningCallback/callback.wait()@resource://services-common/async.js:156 < onNotify()@resource://gre/modules/services-sync/service.js:1009 < WrappedNotify()@resource://services-sync/util.js:148 < WrappedLock()@resource://services-sync/util.js:103 < WrappedCatch()@resource://services-sync/util.js:77 < login()@resource://gre/modules/services-sync/service.js:1031 < sync/<()@resource://gre/modules/services-sync/service.js:1283 < WrappedCatch()@resource://services-sync/util.js:77 < sync()@resource://gre/modules/services-sync/service.js:1279
1566306771246	Hawk	DEBUG	(Response) /recovery_email/status: code: null - Status text: null
1566306771247	Sync.RESTResponse	TRACE	Processing response headers.
1566306771247	Sync.RESTResponse	DEBUG	Caught exception processing response headers:[Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.visitResponseHeaders]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: RESTResponse.prototype.headers :: line 693"  data: no] Stack trace: RESTResponse.prototype.headers()@resource://services-common/rest.js:693 < this.HawkClient.prototype._maybeNotifyBackoff()@resource://services-common/hawkclient.js:315 < _onComplete()@resource://services-common/hawkclient.js:228 < onComplete()@resource://services-common/hawkclient.js:275 < onStopRequest()@resource://services-common/rest.js:467 < waitForSyncCallback()@resource://services-common/async.js:102 < makeSpinningCallback/callback.wait()@resource://services-common/async.js:156 < onNotify()@resource://gre/modules/services-sync/service.js:1009 < WrappedNotify()@resource://services-sync/util.js:148 < WrappedLock()@resource://services-sync/util.js:103 < WrappedCatch()@resource://services-sync/util.js:77 < login()@resource://gre/modules/services-sync/service.js:1031 < sync/<()@resource://gre/modules/services-sync/service.js:1283 < WrappedCatch()@resource://services-sync/util.js:77 < sync()@resource://gre/modules/services-sync/service.js:1279
1566306771247	Sync.RESTResponse	TRACE	Processing response headers.
1566306771248	Sync.RESTResponse	DEBUG	Caught exception processing response headers:[Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.visitResponseHeaders]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: RESTResponse.prototype.headers :: line 693"  data: no] Stack trace: RESTResponse.prototype.headers()@resource://services-common/rest.js:693 < this.HawkClient.prototype._maybeNotifyBackoff()@resource://services-common/hawkclient.js:315 < _onComplete()@resource://services-common/hawkclient.js:229 < onComplete()@resource://services-common/hawkclient.js:275 < onStopRequest()@resource://services-common/rest.js:467 < waitForSyncCallback()@resource://services-common/async.js:102 < makeSpinningCallback/callback.wait()@resource://services-common/async.js:156 < onNotify()@resource://gre/modules/services-sync/service.js:1009 < WrappedNotify()@resource://services-sync/util.js:148 < WrappedLock()@resource://services-sync/util.js:103 < WrappedCatch()@resource://services-sync/util.js:77 < login()@resource://gre/modules/services-sync/service.js:1031 < sync/<()@resource://gre/modules/services-sync/service.js:1283 < WrappedCatch()@resource://services-sync/util.js:77 < sync()@resource://gre/modules/services-sync/service.js:1279
1566306771249	Sync.RESTResponse	DEBUG	Caught exception fetching HTTP status text:[Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatusText]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: RESTResponse.prototype.statusText :: line 659"  data: no] Stack trace: RESTResponse.prototype.statusText()@resource://services-common/rest.js:659 < this.HawkClient.prototype._constructError()@resource://services-common/hawkclient.js:121 < _onComplete()@resource://services-common/hawkclient.js:234 < onComplete()@resource://services-common/hawkclient.js:275 < onStopRequest()@resource://services-common/rest.js:467 < waitForSyncCallback()@resource://services-common/async.js:102 < makeSpinningCallback/callback.wait()@resource://services-common/async.js:156 < onNotify()@resource://gre/modules/services-sync/service.js:1009 < WrappedNotify()@resource://services-sync/util.js:148 < WrappedLock()@resource://services-sync/util.js:103 < WrappedCatch()@resource://services-sync/util.js:77 < login()@resource://gre/modules/services-sync/service.js:1031 < sync/<()@resource://gre/modules/services-sync/service.js:1283 < WrappedCatch()@resource://services-sync/util.js:77 < sync()@resource://gre/modules/services-sync/service.js:1279
1566306771250	Sync.RESTResponse	DEBUG	Caught exception fetching HTTP status code:[Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatus]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: RESTResponse.prototype.status :: line 643"  data: no] Stack trace: RESTResponse.prototype.status()@resource://services-common/rest.js:643 < this.HawkClient.prototype._constructError()@resource://services-common/hawkclient.js:122 < _onComplete()@resource://services-common/hawkclient.js:234 < onComplete()@resource://services-common/hawkclient.js:275 < onStopRequest()@resource://services-common/rest.js:467 < waitForSyncCallback()@resource://services-common/async.js:102 < makeSpinningCallback/callback.wait()@resource://services-common/async.js:156 < onNotify()@resource://gre/modules/services-sync/service.js:1009 < WrappedNotify()@resource://services-sync/util.js:148 < WrappedLock()@resource://services-sync/util.js:103 < WrappedCatch()@resource://services-sync/util.js:77 < login()@resource://gre/modules/services-sync/service.js:1031 < sync/<()@resource://gre/modules/services-sync/service.js:1283 < WrappedCatch()@resource://services-sync/util.js:77 < sync()@resource://gre/modules/services-sync/service.js:1279
1566306771251	Sync.RESTResponse	DEBUG	Caught exception fetching HTTP status code:[Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatus]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: RESTResponse.prototype.status :: line 643"  data: no] Stack trace: RESTResponse.prototype.status()@resource://services-common/rest.js:643 < this.HawkClient.prototype._constructError()@resource://services-common/hawkclient.js:123 < _onComplete()@resource://services-common/hawkclient.js:234 < onComplete()@resource://services-common/hawkclient.js:275 < onStopRequest()@resource://services-common/rest.js:467 < waitForSyncCallback()@resource://services-common/async.js:102 < makeSpinningCallback/callback.wait()@resource://services-common/async.js:156 < onNotify()@resource://gre/modules/services-sync/service.js:1009 < WrappedNotify()@resource://services-sync/util.js:148 < WrappedLock()@resource://services-sync/util.js:103 < WrappedCatch()@resource://services-sync/util.js:77 < login()@resource://gre/modules/services-sync/service.js:1031 < sync/<()@resource://gre/modules/services-sync/service.js:1283 < WrappedCatch()@resource://services-sync/util.js:77 < sync()@resource://gre/modules/services-sync/service.js:1279
1566306771251	Sync.RESTResponse	TRACE	Processing response headers.
1566306771251	Sync.RESTResponse	DEBUG	Caught exception processing response headers:[Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.visitResponseHeaders]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: RESTResponse.prototype.headers :: line 693"  data: no] Stack trace: RESTResponse.prototype.headers()@resource://services-common/rest.js:693 < this.HawkClient.prototype._constructError()@resource://services-common/hawkclient.js:128 < _onComplete()@resource://services-common/hawkclient.js:234 < onComplete()@resource://services-common/hawkclient.js:275 < onStopRequest()@resource://services-common/rest.js:467 < waitForSyncCallback()@resource://services-common/async.js:102 < makeSpinningCallback/callback.wait()@resource://services-common/async.js:156 < onNotify()@resource://gre/modules/services-sync/service.js:1009 < WrappedNotify()@resource://services-sync/util.js:148 < WrappedLock()@resource://services-sync/util.js:103 < WrappedCatch()@resource://services-sync/util.js:77 < login()@resource://gre/modules/services-sync/service.js:1031 < sync/<()@resource://gre/modules/services-sync/service.js:1283 < WrappedCatch()@resource://services-sync/util.js:77 < sync()@resource://gre/modules/services-sync/service.js:1279
1566306771252	FirefoxAccounts	ERROR	error GETing /recovery_email/status: {"error":{},"errorString":"[Exception... \"NS_ERROR_UNKNOWN_HOST\"  nsresult: \"0x804b001e (NS_ERROR_UNKNOWN_HOST)\"  location: \"JS frame :: resource://services-common/rest.js :: onStopRequest :: line 465\"  data: no]","message":null,"code":null,"errno":null}
1566306771253	FirefoxAccounts	DEBUG	polling with timeout = 5000
1566306776254	FirefoxAccounts	DEBUG	entering pollEmailStatus: timer
1566306776266	Hawk	WARN	hawk request error: [Exception... "NS_ERROR_UNKNOWN_HOST"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "JS frame :: resource://services-common/rest.js :: onStopRequest :: line 465"  data: no] Stack trace: onStopRequest()@resource://services-common/rest.js:465 < waitForSyncCallback()@resource://services-common/async.js:102 < makeSpinningCallback/callback.wait()@resource://services-common/async.js:156 < onNotify()@resource://gre/modules/services-sync/service.js:1009 < WrappedNotify()@resource://services-sync/util.js:148 < WrappedLock()@resource://services-sync/util.js:103 < WrappedCatch()@resource://services-sync/util.js:77 < login()@resource://gre/modules/services-sync/service.js:1031 < sync/<()@resource://gre/modules/services-sync/service.js:1283 < WrappedCatch()@resource://services-sync/util.js:77 < sync()@resource://gre/modules/services-sync/service.js:1279
1566306776267	Sync.RESTResponse	DEBUG	Caught exception fetching HTTP status code:[Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatus]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: RESTResponse.prototype.status :: line 643"  data: no] Stack trace: RESTResponse.prototype.status()@resource://services-common/rest.js:643 < _onComplete()@resource://services-common/hawkclient.js:218 < onComplete()@resource://services-common/hawkclient.js:275 < onStopRequest()@resource://services-common/rest.js:467 < waitForSyncCallback()@resource://services-common/async.js:102 < makeSpinningCallback/callback.wait()@resource://services-common/async.js:156 < onNotify()@resource://gre/modules/services-sync/service.js:1009 < WrappedNotify()@resource://services-sync/util.js:148 < WrappedLock()@resource://services-sync/util.js:103 < WrappedCatch()@resource://services-sync/util.js:77 < login()@resource://gre/modules/services-sync/service.js:1031 < sync/<()@resource://gre/modules/services-sync/service.js:1283 < WrappedCatch()@resource://services-sync/util.js:77 < sync()@resource://gre/modules/services-sync/service.js:1279
1566306776268	Sync.RESTResponse	DEBUG	Caught exception fetching HTTP status text:[Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatusText]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: RESTResponse.prototype.statusText :: line 659"  data: no] Stack trace: RESTResponse.prototype.statusText()@resource://services-common/rest.js:659 < _onComplete()@resource://services-common/hawkclient.js:221 < onComplete()@resource://services-common/hawkclient.js:275 < onStopRequest()@resource://services-common/rest.js:467 < waitForSyncCallback()@resource://services-common/async.js:102 < makeSpinningCallback/callback.wait()@resource://services-common/async.js:156 < onNotify()@resource://gre/modules/services-sync/service.js:1009 < WrappedNotify()@resource://services-sync/util.js:148 < WrappedLock()@resource://services-sync/util.js:103 < WrappedCatch()@resource://services-sync/util.js:77 < login()@resource://gre/modules/services-sync/service.js:1031 < sync/<()@resource://gre/modules/services-sync/service.js:1283 < WrappedCatch()@resource://services-sync/util.js:77 < sync()@resource://gre/modules/services-sync/service.js:1279
1566306776268	Hawk	DEBUG	(Response) /recovery_email/status: code: null - Status text: null
1566306776269	Sync.RESTResponse	TRACE	Processing response headers.
1566306776269	Sync.RESTResponse	DEBUG	Caught exception processing response headers:[Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.visitResponseHeaders]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: RESTResponse.prototype.headers :: line 693"  data: no] Stack trace: RESTResponse.prototype.headers()@resource://services-common/rest.js:693 < this.HawkClient.prototype._maybeNotifyBackoff()@resource://services-common/hawkclient.js:315 < _onComplete()@resource://services-common/hawkclient.js:228 < onComplete()@resource://services-common/hawkclient.js:275 < onStopRequest()@resource://services-common/rest.js:467 < waitForSyncCallback()@resource://services-common/async.js:102 < makeSpinningCallback/callback.wait()@resource://services-common/async.js:156 < onNotify()@resource://gre/modules/services-sync/service.js:1009 < WrappedNotify()@resource://services-sync/util.js:148 < WrappedLock()@resource://services-sync/util.js:103 < WrappedCatch()@resource://services-sync/util.js:77 < login()@resource://gre/modules/services-sync/service.js:1031 < sync/<()@resource://gre/modules/services-sync/service.js:1283 < WrappedCatch()@resource://services-sync/util.js:77 < sync()@resource://gre/modules/services-sync/service.js:1279
1566306776270	Sync.RESTResponse	TRACE	Processing response headers.
1566306776270	Sync.RESTResponse	DEBUG	Caught exception processing response headers:[Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.visitResponseHeaders]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: RESTResponse.prototype.headers :: line 693"  data: no] Stack trace: RESTResponse.prototype.headers()@resource://services-common/rest.js:693 < this.HawkClient.prototype._maybeNotifyBackoff()@resource://services-common/hawkclient.js:315 < _onComplete()@resource://services-common/hawkclient.js:229 < onComplete()@resource://services-common/hawkclient.js:275 < onStopRequest()@resource://services-common/rest.js:467 < waitForSyncCallback()@resource://services-common/async.js:102 < makeSpinningCallback/callback.wait()@resource://services-common/async.js:156 < onNotify()@resource://gre/modules/services-sync/service.js:1009 < WrappedNotify()@resource://services-sync/util.js:148 < WrappedLock()@resource://services-sync/util.js:103 < WrappedCatch()@resource://services-sync/util.js:77 < login()@resource://gre/modules/services-sync/service.js:1031 < sync/<()@resource://gre/modules/services-sync/service.js:1283 < WrappedCatch()@resource://services-sync/util.js:77 < sync()@resource://gre/modules/services-sync/service.js:1279
1566306776271	Sync.RESTResponse	DEBUG	Caught exception fetching HTTP status text:[Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatusText]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: RESTResponse.prototype.statusText :: line 659"  data: no] Stack trace: RESTResponse.prototype.statusText()@resource://services-common/rest.js:659 < this.HawkClient.prototype._constructError()@resource://services-common/hawkclient.js:121 < _onComplete()@resource://services-common/hawkclient.js:234 < onComplete()@resource://services-common/hawkclient.js:275 < onStopRequest()@resource://services-common/rest.js:467 < waitForSyncCallback()@resource://services-common/async.js:102 < makeSpinningCallback/callback.wait()@resource://services-common/async.js:156 < onNotify()@resource://gre/modules/services-sync/service.js:1009 < WrappedNotify()@resource://services-sync/util.js:148 < WrappedLock()@resource://services-sync/util.js:103 < WrappedCatch()@resource://services-sync/util.js:77 < login()@resource://gre/modules/services-sync/service.js:1031 < sync/<()@resource://gre/modules/services-sync/service.js:1283 < WrappedCatch()@resource://services-sync/util.js:77 < sync()@resource://gre/modules/services-sync/service.js:1279
1566306776272	Sync.RESTResponse	DEBUG	Caught exception fetching HTTP status code:[Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatus]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: RESTResponse.prototype.status :: line 643"  data: no] Stack trace: RESTResponse.prototype.status()@resource://services-common/rest.js:643 < this.HawkClient.prototype._constructError()@resource://services-common/hawkclient.js:122 < _onComplete()@resource://services-common/hawkclient.js:234 < onComplete()@resource://services-common/hawkclient.js:275 < onStopRequest()@resource://services-common/rest.js:467 < waitForSyncCallback()@resource://services-common/async.js:102 < makeSpinningCallback/callback.wait()@resource://services-common/async.js:156 < onNotify()@resource://gre/modules/services-sync/service.js:1009 < WrappedNotify()@resource://services-sync/util.js:148 < WrappedLock()@resource://services-sync/util.js:103 < WrappedCatch()@resource://services-sync/util.js:77 < login()@resource://gre/modules/services-sync/service.js:1031 < sync/<()@resource://gre/modules/services-sync/service.js:1283 < WrappedCatch()@resource://services-sync/util.js:77 < sync()@resource://gre/modules/services-sync/service.js:1279
1566306776273	Sync.RESTResponse	DEBUG	Caught exception fetching HTTP status code:[Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatus]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: RESTResponse.prototype.status :: line 643"  data: no] Stack trace: RESTResponse.prototype.status()@resource://services-common/rest.js:643 < this.HawkClient.prototype._constructError()@resource://services-common/hawkclient.js:123 < _onComplete()@resource://services-common/hawkclient.js:234 < onComplete()@resource://services-common/hawkclient.js:275 < onStopRequest()@resource://services-common/rest.js:467 < waitForSyncCallback()@resource://services-common/async.js:102 < makeSpinningCallback/callback.wait()@resource://services-common/async.js:156 < onNotify()@resource://gre/modules/services-sync/service.js:1009 < WrappedNotify()@resource://services-sync/util.js:148 < WrappedLock()@resource://services-sync/util.js:103 < WrappedCatch()@resource://services-sync/util.js:77 < login()@resource://gre/modules/services-sync/service.js:1031 < sync/<()@resource://gre/modules/services-sync/service.js:1283 < WrappedCatch()@resource://services-sync/util.js:77 < sync()@resource://gre/modules/services-sync/service.js:1279
1566306776273	Sync.RESTResponse	TRACE	Processing response headers.
1566306776274	Sync.RESTResponse	DEBUG	Caught exception processing response headers:[Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.visitResponseHeaders]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: RESTResponse.prototype.headers :: line 693"  data: no] Stack trace: RESTResponse.prototype.headers()@resource://services-common/rest.js:693 < this.HawkClient.prototype._constructError()@resource://services-common/hawkclient.js:128 < _onComplete()@resource://services-common/hawkclient.js:234 < onComplete()@resource://services-common/hawkclient.js:275 < onStopRequest()@resource://services-common/rest.js:467 < waitForSyncCallback()@resource://services-common/async.js:102 < makeSpinningCallback/callback.wait()@resource://services-common/async.js:156 < onNotify()@resource://gre/modules/services-sync/service.js:1009 < WrappedNotify()@resource://services-sync/util.js:148 < WrappedLock()@resource://services-sync/util.js:103 < WrappedCatch()@resource://services-sync/util.js:77 < login()@resource://gre/modules/services-sync/service.js:1031 < sync/<()@resource://gre/modules/services-sync/service.js:1283 < WrappedCatch()@resource://services-sync/util.js:77 < sync()@resource://gre/modules/services-sync/service.js:1279
1566306776274	FirefoxAccounts	ERROR	error GETing /recovery_email/status: {"error":{},"errorString":"[Exception... \"NS_ERROR_UNKNOWN_HOST\"  nsresult: \"0x804b001e (NS_ERROR_UNKNOWN_HOST)\"  location: \"JS frame :: resource://services-common/rest.js :: onStopRequest :: line 465\"  data: no]","message":null,"code":null,"errno":null}
1566306776275	FirefoxAccounts	DEBUG	polling with timeout = 5000
1566306781276	FirefoxAccounts	DEBUG	entering pollEmailStatus: timer
1566306781287	Hawk	WARN	hawk request error: [Exception... "NS_ERROR_UNKNOWN_HOST"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "JS frame :: resource://services-common/rest.js :: onStopRequest :: line 465"  data: no] Stack trace: onStopRequest()@resource://services-common/rest.js:465 < waitForSyncCallback()@resource://services-common/async.js:102 < makeSpinningCallback/callback.wait()@resource://services-common/async.js:156 < onNotify()@resource://gre/modules/services-sync/service.js:1009 < WrappedNotify()@resource://services-sync/util.js:148 < WrappedLock()@resource://services-sync/util.js:103 < WrappedCatch()@resource://services-sync/util.js:77 < login()@resource://gre/modules/services-sync/service.js:1031 < sync/<()@resource://gre/modules/services-sync/service.js:1283 < WrappedCatch()@resource://services-sync/util.js:77 < sync()@resource://gre/modules/services-sync/service.js:1279
1566306781293	Sync.RESTResponse	DEBUG	Caught exception fetching HTTP status code:[Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatus]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: RESTResponse.prototype.status :: line 643"  data: no] Stack trace: RESTResponse.prototype.status()@resource://services-common/rest.js:643 < _onComplete()@resource://services-common/hawkclient.js:218 < onComplete()@resource://services-common/hawkclient.js:275 < onStopRequest()@resource://services-common/rest.js:467 < waitForSyncCallback()@resource://services-common/async.js:102 < makeSpinningCallback/callback.wait()@resource://services-common/async.js:156 < onNotify()@resource://gre/modules/services-sync/service.js:1009 < WrappedNotify()@resource://services-sync/util.js:148 < WrappedLock()@resource://services-sync/util.js:103 < WrappedCatch()@resource://services-sync/util.js:77 < login()@resource://gre/modules/services-sync/service.js:1031 < sync/<()@resource://gre/modules/services-sync/service.js:1283 < WrappedCatch()@resource://services-sync/util.js:77 < sync()@resource://gre/modules/services-sync/service.js:1279
1566306781293	Sync.RESTResponse	DEBUG	Caught exception fetching HTTP status text:[Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatusText]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: RESTResponse.prototype.statusText :: line 659"  data: no] Stack trace: RESTResponse.prototype.statusText()@resource://services-common/rest.js:659 < _onComplete()@resource://services-common/hawkclient.js:221 < onComplete()@resource://services-common/hawkclient.js:275 < onStopRequest()@resource://services-common/rest.js:467 < waitForSyncCallback()@resource://services-common/async.js:102 < makeSpinningCallback/callback.wait()@resource://services-common/async.js:156 < onNotify()@resource://gre/modules/services-sync/service.js:1009 < WrappedNotify()@resource://services-sync/util.js:148 < WrappedLock()@resource://services-sync/util.js:103 < WrappedCatch()@resource://services-sync/util.js:77 < login()@resource://gre/modules/services-sync/service.js:1031 < sync/<()@resource://gre/modules/services-sync/service.js:1283 < WrappedCatch()@resource://services-sync/util.js:77 < sync()@resource://gre/modules/services-sync/service.js:1279
1566306781293	Hawk	DEBUG	(Response) /recovery_email/status: code: null - Status text: null
1566306781293	Sync.RESTResponse	TRACE	Processing response headers.
1566306781294	Sync.RESTResponse	DEBUG	Caught exception processing response headers:[Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.visitResponseHeaders]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: RESTResponse.prototype.headers :: line 693"  data: no] Stack trace: RESTResponse.prototype.headers()@resource://services-common/rest.js:693 < this.HawkClient.prototype._maybeNotifyBackoff()@resource://services-common/hawkclient.js:315 < _onComplete()@resource://services-common/hawkclient.js:228 < onComplete()@resource://services-common/hawkclient.js:275 < onStopRequest()@resource://services-common/rest.js:467 < waitForSyncCallback()@resource://services-common/async.js:102 < makeSpinningCallback/callback.wait()@resource://services-common/async.js:156 < onNotify()@resource://gre/modules/services-sync/service.js:1009 < WrappedNotify()@resource://services-sync/util.js:148 < WrappedLock()@resource://services-sync/util.js:103 < WrappedCatch()@resource://services-sync/util.js:77 < login()@resource://gre/modules/services-sync/service.js:1031 < sync/<()@resource://gre/modules/services-sync/service.js:1283 < WrappedCatch()@resource://services-sync/util.js:77 < sync()@resource://gre/modules/services-sync/service.js:1279
1566306781294	Sync.RESTResponse	TRACE	Processing response headers.
1566306781295	Sync.RESTResponse	DEBUG	Caught exception processing response headers:[Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.visitResponseHeaders]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: RESTResponse.prototype.headers :: line 693"  data: no] Stack trace: RESTResponse.prototype.headers()@resource://services-common/rest.js:693 < this.HawkClient.prototype._maybeNotifyBackoff()@resource://services-common/hawkclient.js:315 < _onComplete()@resource://services-common/hawkclient.js:229 < onComplete()@resource://services-common/hawkclient.js:275 < onStopRequest()@resource://services-common/rest.js:467 < waitForSyncCallback()@resource://services-common/async.js:102 < makeSpinningCallback/callback.wait()@resource://services-common/async.js:156 < onNotify()@resource://gre/modules/services-sync/service.js:1009 < WrappedNotify()@resource://services-sync/util.js:148 < WrappedLock()@resource://services-sync/util.js:103 < WrappedCatch()@resource://services-sync/util.js:77 < login()@resource://gre/modules/services-sync/service.js:1031 < sync/<()@resource://gre/modules/services-sync/service.js:1283 < WrappedCatch()@resource://services-sync/util.js:77 < sync()@resource://gre/modules/services-sync/service.js:1279
1566306781296	Sync.RESTResponse	DEBUG	Caught exception fetching HTTP status text:[Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatusText]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: RESTResponse.prototype.statusText :: line 659"  data: no] Stack trace: RESTResponse.prototype.statusText()@resource://services-common/rest.js:659 < this.HawkClient.prototype._constructError()@resource://services-common/hawkclient.js:121 < _onComplete()@resource://services-common/hawkclient.js:234 < onComplete()@resource://services-common/hawkclient.js:275 < onStopRequest()@resource://services-common/rest.js:467 < waitForSyncCallback()@resource://services-common/async.js:102 < makeSpinningCallback/callback.wait()@resource://services-common/async.js:156 < onNotify()@resource://gre/modules/services-sync/service.js:1009 < WrappedNotify()@resource://services-sync/util.js:148 < WrappedLock()@resource://services-sync/util.js:103 < WrappedCatch()@resource://services-sync/util.js:77 < login()@resource://gre/modules/services-sync/service.js:1031 < sync/<()@resource://gre/modules/services-sync/service.js:1283 < WrappedCatch()@resource://services-sync/util.js:77 < sync()@resource://gre/modules/services-sync/service.js:1279
1566306781305	Sync.RESTResponse	DEBUG	Caught exception fetching HTTP status code:[Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatus]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: RESTResponse.prototype.status :: line 643"  data: no] Stack trace: RESTResponse.prototype.status()@resource://services-common/rest.js:643 < this.HawkClient.prototype._constructError()@resource://services-common/hawkclient.js:122 < _onComplete()@resource://services-common/hawkclient.js:234 < onComplete()@resource://services-common/hawkclient.js:275 < onStopRequest()@resource://services-common/rest.js:467 < waitForSyncCallback()@resource://services-common/async.js:102 < makeSpinningCallback/callback.wait()@resource://services-common/async.js:156 < onNotify()@resource://gre/modules/services-sync/service.js:1009 < WrappedNotify()@resource://services-sync/util.js:148 < WrappedLock()@resource://services-sync/util.js:103 < WrappedCatch()@resource://services-sync/util.js:77 < login()@resource://gre/modules/services-sync/service.js:1031 < sync/<()@resource://gre/modules/services-sync/service.js:1283 < WrappedCatch()@resource://services-sync/util.js:77 < sync()@resource://gre/modules/services-sync/service.js:1279
1566306781306	Sync.RESTResponse	DEBUG	Caught exception fetching HTTP status code:[Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatus]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: RESTResponse.prototype.status :: line 643"  data: no] Stack trace: RESTResponse.prototype.status()@resource://services-common/rest.js:643 < this.HawkClient.prototype._constructError()@resource://services-common/hawkclient.js:123 < _onComplete()@resource://services-common/hawkclient.js:234 < onComplete()@resource://services-common/hawkclient.js:275 < onStopRequest()@resource://services-common/rest.js:467 < waitForSyncCallback()@resource://services-common/async.js:102 < makeSpinningCallback/callback.wait()@resource://services-common/async.js:156 < onNotify()@resource://gre/modules/services-sync/service.js:1009 < WrappedNotify()@resource://services-sync/util.js:148 < WrappedLock()@resource://services-sync/util.js:103 < WrappedCatch()@resource://services-sync/util.js:77 < login()@resource://gre/modules/services-sync/service.js:1031 < sync/<()@resource://gre/modules/services-sync/service.js:1283 < WrappedCatch()@resource://services-sync/util.js:77 < sync()@resource://gre/modules/services-sync/service.js:1279
1566306781306	Sync.RESTResponse	TRACE	Processing response headers.
1566306781310	Sync.RESTResponse	DEBUG	Caught exception processing response headers:[Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.visitResponseHeaders]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: RESTResponse.prototype.headers :: line 693"  data: no] Stack trace: RESTResponse.prototype.headers()@resource://services-common/rest.js:693 < this.HawkClient.prototype._constructError()@resource://services-common/hawkclient.js:128 < _onComplete()@resource://services-common/hawkclient.js:234 < onComplete()@resource://services-common/hawkclient.js:275 < onStopRequest()@resource://services-common/rest.js:467 < waitForSyncCallback()@resource://services-common/async.js:102 < makeSpinningCallback/callback.wait()@resource://services-common/async.js:156 < onNotify()@resource://gre/modules/services-sync/service.js:1009 < WrappedNotify()@resource://services-sync/util.js:148 < WrappedLock()@resource://services-sync/util.js:103 < WrappedCatch()@resource://services-sync/util.js:77 < login()@resource://gre/modules/services-sync/service.js:1031 < sync/<()@resource://gre/modules/services-sync/service.js:1283 < WrappedCatch()@resource://services-sync/util.js:77 < sync()@resource://gre/modules/services-sync/service.js:1279
1566306781311	FirefoxAccounts	ERROR	error GETing /recovery_email/status: {"error":{},"errorString":"[Exception... \"NS_ERROR_UNKNOWN_HOST\"  nsresult: \"0x804b001e (NS_ERROR_UNKNOWN_HOST)\"  location: \"JS frame :: resource://services-common/rest.js :: onStopRequest :: line 465\"  data: no]","message":null,"code":null,"errno":null}
1566306781313	FirefoxAccounts	DEBUG	polling with timeout = 5000
1566306786313	FirefoxAccounts	DEBUG	entering pollEmailStatus: timer
1566306797765	Hawk	DEBUG	(Response) /recovery_email/status: code: 200 - Status text: OK
1566306797766	Sync.RESTResponse	TRACE	Processing response headers.
1566306797767	Hawk	DEBUG	Clock offset vs https://api.accounts.firefox.com/v1: -767
1566306797768	FirefoxAccounts	DEBUG	checkEmailStatus -> {"email":"ishayananbal6@gmail.com","verified":true,"sessionVerified":true,"emailVerified":true}
1566306797769	FirefoxAccounts	DEBUG	_updateAccountData with items: ["verified"]
1566306797774	FirefoxAccounts	DEBUG	Notifying observers of fxaccounts:update
1566306797780	Sync.BrowserIDManager	INFO	Starting fetch for key bundle.
1566306797786	FirefoxAccounts	DEBUG	already verified
1566306797788	Sync.BrowserIDManager	INFO	Fetching new keys
1566306797790	FirefoxAccounts	DEBUG	fetchKeys: true
1566306797797	FirefoxAccounts	DEBUG	writing plain storage: ["email","sessionToken","uid","verified"]
1566306797798	FirefoxAccounts	TRACE	starting write of json user data: ["email","sessionToken","uid","verified"]
1566306797806	FirefoxAccounts	TRACE	finished write of json user data - took: 8
1566306797807	FirefoxAccounts	DEBUG	writing secure storage: ["keyFetchToken","unwrapBKey"]
1566306797807	FirefoxAccounts	TRACE	starting write of user data to the login manager
1566306797818	FirefoxAccounts	TRACE	finished write of user data to the login manager
1566306798541	Hawk	DEBUG	(Response) /account/keys: code: 200 - Status text: OK
1566306798542	Sync.RESTResponse	TRACE	Processing response headers.
1566306798543	Hawk	DEBUG	Clock offset vs https://api.accounts.firefox.com/v1: -543
1566306798549	FirefoxAccounts	DEBUG	Keys Obtained: kA=true, kB=true
1566306798550	FirefoxAccounts	DEBUG	_updateAccountData with items: ["kA","kB","keyFetchToken","unwrapBKey"]
1566306798552	FirefoxAccounts	DEBUG	Notifying observers of fxaccounts:onverified
1566306798561	Sync.BrowserIDManager	INFO	Getting an assertion from: https://token.services.mozilla.com/1.0/sync/1.5
1566306798562	FirefoxAccounts	DEBUG	enter getAssertion()
1566306798586	FirefoxAccounts	DEBUG	writing plain storage: ["email","sessionToken","uid","verified"]
1566306798587	FirefoxAccounts	TRACE	starting write of json user data: ["email","sessionToken","uid","verified"]
1566306798593	FirefoxAccounts	DEBUG	got keyPair
1566306798594	FirefoxAccounts	DEBUG	getCertificateSigned: true true
1566306798605	FirefoxAccounts	TRACE	finished write of json user data - took: 18
1566306798606	FirefoxAccounts	DEBUG	writing secure storage: ["kA","kB"]
1566306798606	FirefoxAccounts	TRACE	starting write of user data to the login manager
1566306798616	FirefoxAccounts	TRACE	finished write of user data to the login manager
1566306799235	Hawk	DEBUG	(Response) /certificate/sign: code: 200 - Status text: OK
1566306799235	Sync.RESTResponse	TRACE	Processing response headers.
1566306799236	Hawk	DEBUG	Clock offset vs https://api.accounts.firefox.com/v1: -1236
1566306799237	FirefoxAccounts	DEBUG	getCertificate got a new one: true
1566306799239	FirefoxAccounts	DEBUG	_updateAccountData with items: ["keyPair","cert"]
1566306799240	FirefoxAccounts	DEBUG	getAssertionFromCert
1566306799241	FirefoxAccounts	DEBUG	writing plain storage: ["email","sessionToken","uid","verified"]
1566306799241	FirefoxAccounts	TRACE	starting write of json user data: ["email","sessionToken","uid","verified"]
1566306799251	FirefoxAccounts	TRACE	finished write of json user data - took: 10
1566306799252	FirefoxAccounts	DEBUG	writing secure storage: ["kA","kB"]
1566306799252	FirefoxAccounts	TRACE	starting write of user data to the login manager
1566306799262	FirefoxAccounts	TRACE	finished write of user data to the login manager
1566306799262	FirefoxAccounts	DEBUG	getAssertionFromCert returning signed: true
1566306799263	Sync.BrowserIDManager	DEBUG	Getting a token
1566306799264	Common.TokenServerClient	DEBUG	Beginning BID assertion exchange: https://token.services.mozilla.com/1.0/sync/1.5
1566306860919	Sync.SyncScheduler	DEBUG	Next sync in 3600000 ms.
1566306987641	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1566306987643	Sync.SyncScheduler	DEBUG	Next sync in 3600000 ms.
1566306987644	Sync.ErrorHandler	ERROR	Sync encountered a login error
